# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Red Hat, Inc.
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
# Dimitris Glezos <glezos@transifex.com>, 2011
# Iñigo Varela <ivarela@softastur.org>, 2011
msgid ""
msgstr ""
"Project-Id-Version: anaconda 29.19.2.16\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2020-02-21 09:41+0100\n"
"PO-Revision-Date: 2015-01-05 12:46+0000\n"
"Last-Translator: Copied by Zanata <copied-by-zanata@zanata.org>\n"
"Language-Team: Asturian (http://www.transifex.com/projects/p/anaconda/language/ast/)\n"
"Language: ast\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Zanata 4.6.2\n"

#: anaconda.py:409
msgid "Unable to create PID file"
msgstr ""

#: anaconda.py:410
#, python-format
msgid ""
"Anaconda is unable to create %s because the file already exists. Anaconda is"
" already running, or a previous instance of anaconda has crashed."
msgstr ""

#: pyanaconda/addons.py:90
#, python-format
msgid "Executing %s addon"
msgstr ""

#: pyanaconda/input_checking.py:601
msgid "Full name cannot contain colon characters"
msgstr ""

#: pyanaconda/installation.py:93
msgid "Configuring installed system"
msgstr ""

#: pyanaconda/installation.py:109
msgid "Writing network configuration"
msgstr ""

#: pyanaconda/installation.py:116
msgid "Creating users"
msgstr ""

#: pyanaconda/installation.py:124
msgid "Configuring addons"
msgstr ""

#: pyanaconda/installation.py:129
msgid "Generating initramfs"
msgstr ""

#: pyanaconda/installation.py:151
#, python-format
msgid "Joining realm: %s"
msgstr ""

#: pyanaconda/installation.py:155
msgid "Running post-installation scripts"
msgstr ""

#: pyanaconda/installation.py:161
msgid "Setting up kexec"
msgstr ""

#: pyanaconda/installation.py:166
msgid "Storing configuration files and kickstarts"
msgstr ""

#: pyanaconda/installation.py:243
#, python-format
msgid "Waiting for %s threads to finish"
msgstr ""

#: pyanaconda/installation.py:257
msgid "Setting up the installation environment"
msgstr ""

#: pyanaconda/installation.py:265 pyanaconda/installation.py:361
msgid "Configuring storage"
msgstr ""

#: pyanaconda/installation.py:294 pyanaconda/kickstart.py:2814
msgid "Running pre-installation scripts"
msgstr "Executando los scripts de pre-instalación"

#: pyanaconda/installation.py:301
msgid "Discovering realm to join"
msgstr ""

#: pyanaconda/installation.py:306
msgid "Running pre-installation tasks"
msgstr ""

#: pyanaconda/installation.py:356
msgid "Installing."
msgstr ""

#: pyanaconda/installation.py:366
msgid "Configuring Red Hat subscription"
msgstr ""

#: pyanaconda/installation.py:378
msgid "Installing boot loader"
msgstr ""

#: pyanaconda/installation.py:382 pyanaconda/payload/dnfpayload.py:1091
#: pyanaconda/payload/livepayload.py:203
msgid "Performing post-installation setup tasks"
msgstr ""

#: pyanaconda/installation.py:388
msgid "Creating snapshots"
msgstr ""

#: pyanaconda/storage_utils.py:69 pyanaconda/storage_utils.py:103
#: pyanaconda/ui/gui/spokes/custom_storage.glade:31
msgid "LVM"
msgstr ""

#: pyanaconda/storage_utils.py:70 pyanaconda/storage_utils.py:104
#: pyanaconda/ui/gui/spokes/custom_storage.glade:39
msgid "LVM Thin Provisioning"
msgstr ""

#: pyanaconda/storage_utils.py:71
#: pyanaconda/ui/gui/spokes/custom_storage.glade:35
msgid "RAID"
msgstr ""

#: pyanaconda/storage_utils.py:72 pyanaconda/storage_utils.py:101
#: pyanaconda/ui/gui/spokes/custom_storage.glade:23
msgid "Standard Partition"
msgstr ""

#: pyanaconda/storage_utils.py:73 pyanaconda/storage_utils.py:102
#: pyanaconda/ui/gui/spokes/custom_storage.glade:27
msgid "Btrfs"
msgstr ""

#: pyanaconda/storage_utils.py:74
#: pyanaconda/ui/gui/spokes/lib/resize.glade:152
msgid "Disk"
msgstr ""

#: pyanaconda/storage_utils.py:77
msgid "Unsupported"
msgstr ""

#: pyanaconda/storage_utils.py:88
msgid ""
"The 'swap' area on your computer is used by the operating\n"
"system when running low on memory."
msgstr ""

#: pyanaconda/storage_utils.py:90
msgid ""
"The 'boot' area on your computer is where files needed\n"
"to start the operating system are stored."
msgstr ""

#: pyanaconda/storage_utils.py:92
msgid ""
"The 'root' area on your computer is where core system\n"
"files and applications are stored."
msgstr ""

#: pyanaconda/storage_utils.py:94
msgid ""
"The 'home' area on your computer is where all your personal\n"
"data is stored."
msgstr ""

#: pyanaconda/storage_utils.py:96
msgid ""
"The BIOS boot partition is required to enable booting\n"
"from GPT-partitioned disks on BIOS hardware."
msgstr ""

#: pyanaconda/storage_utils.py:98
msgid ""
"The PReP boot partition is required as part of the\n"
"boot loader configuration on some PPC platforms."
msgstr ""

#: pyanaconda/storage_utils.py:178
#, python-format
msgid ""
"Your root partition is less than %(size)s which is usually too small to "
"install %(product)s."
msgstr ""

#: pyanaconda/storage_utils.py:184
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""

#: pyanaconda/storage_utils.py:197
#, python-format
msgid "Your root partition must be on a device of type: %s."
msgstr ""

#: pyanaconda/storage_utils.py:224
msgid ""
"This platform requires /boot on a dedicated partition or logical volume. If "
"you do not want a /boot volume, you must place / on a dedicated non-LVM "
"partition."
msgstr ""

#: pyanaconda/storage_utils.py:231
#, python-brace-format
msgid ""
"The LDL DASD disk {name} ({busid}) cannot be used for the installation. "
"Please format it."
msgstr ""

#: pyanaconda/storage_utils.py:248
#, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""

#: pyanaconda/storage_utils.py:256
#, python-format
msgid "Your %(mount)s partition size is lower than required %(size)s."
msgstr ""

#: pyanaconda/storage_utils.py:276
#, python-format
msgid ""
"Your %(mount)s partition is too small for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""

#: pyanaconda/storage_utils.py:282
#, python-format
msgid ""
"Your %(mount)s partition is too large for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""

#: pyanaconda/storage_utils.py:300
msgid "No valid boot loader target device found. See below for details."
msgstr ""

#: pyanaconda/storage_utils.py:315
msgid "You have not created a bootable partition."
msgstr ""

#: pyanaconda/storage_utils.py:350
msgid ""
"Your BIOS-based system needs a special partition to boot from a GPT disk "
"label. To continue, please create a 1MiB 'biosboot' type partition."
msgstr ""

#: pyanaconda/storage_utils.py:372
#, python-format
msgid ""
"You have not specified a swap partition. %(requiredMem)s of memory is "
"recommended to continue installation without a swap partition, but you only "
"have %(installedMem)s."
msgstr ""

#: pyanaconda/storage_utils.py:379
#, python-format
msgid ""
"You have not specified a swap partition. %(requiredMem)s of memory is "
"required to continue installation without a swap partition, but you only "
"have %(installedMem)s."
msgstr ""

#: pyanaconda/storage_utils.py:385
msgid ""
"You have not specified a swap partition. Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""

#: pyanaconda/storage_utils.py:403
msgid ""
"At least one of your swap devices does not have a UUID, which is common in "
"swap space created using older versions of mkswap. These devices will be "
"referred to by device path in /etc/fstab, which is not ideal since device "
"paths can change under a variety of circumstances. "
msgstr ""

#: pyanaconda/storage_utils.py:422
#, python-format
msgid ""
"This mount point is invalid. The %s directory must be on the / file system."
msgstr ""

#: pyanaconda/storage_utils.py:438
#, python-format
msgid "Your %s must be on a separate partition or LV."
msgstr ""

#: pyanaconda/storage_utils.py:455
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr ""

#: pyanaconda/storage_utils.py:474
#, python-format
msgid ""
"Encryption requested for LUKS device %s but no encryption key specified for "
"this device."
msgstr ""

#: pyanaconda/storage_utils.py:495
#, python-format
msgid ""
"The available memory is less than %(size)s which can be too small for LUKS2 "
"format. It may fail."
msgstr ""

#: pyanaconda/storage_utils.py:518
#, python-format
msgid ""
"%s is currently mounted and cannot be used for the installation. Please "
"unmount it and retry."
msgstr ""

#: pyanaconda/display.py:287
msgid "Graphical installation is not available. Starting text mode."
msgstr "La instalación gráfica nun ta disponible. Aniciando en mou testu."

#: pyanaconda/display.py:295
msgid ""
"Text mode provides a limited set of installation options. It does not offer "
"custom partitioning for full control over the disk layout. Would you like to"
" use VNC mode instead?"
msgstr ""

#: pyanaconda/display.py:324
msgid ""
"X was unable to start on your machine. Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"installation or continue with a text mode installation?"
msgstr ""

#: pyanaconda/errors.py:112
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"The installation will now terminate."
msgstr ""

#: pyanaconda/errors.py:118
#, python-format
msgid "An error occurred while resizing the device %s."
msgstr ""

#: pyanaconda/errors.py:127
#, python-format
msgid ""
"There is a problem with your existing storage configuration: %(errortxt)s\n"
"\n"
"You must resolve this matter before the installation can proceed. There is a shell available for use which you can access by pressing ctrl-alt-f1 and then ctrl-b 2.\n"
"\n"
"Once you have resolved the issue you can retry the storage scan. If you do not fix it you will have to exit the installer."
msgstr ""

#: pyanaconda/errors.py:136
msgctxt "GUI|Storage Detailed Error Dialog"
msgid "_Exit Installer"
msgstr ""

#: pyanaconda/errors.py:137
msgctxt "GUI|Storage Detailed Error Dialog"
msgid "_Retry"
msgstr ""

#: pyanaconda/errors.py:144
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems.  Please check your hardware for the cause of this problem."
msgstr ""

#: pyanaconda/errors.py:153
msgid ""
"There is an entry in your /etc/fstab file that contains an invalid or incorrect file system type:\n"
"\n"
msgstr ""

#: pyanaconda/errors.py:159
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can choose to continue if you think this is in error. Would you like to continue using this image?"
msgstr ""

#: pyanaconda/errors.py:173
msgid ""
"The installer has tried to mount the installation image, but cannot find it on the hard drive.\n"
"\n"
"Should I try again to locate the image?"
msgstr ""

#: pyanaconda/errors.py:184
#, python-format
msgid ""
"The group '%s' is required for this installation. This group does not exist."
" This is a fatal error and installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:190
#, python-format
msgid ""
"You have specified that the group '%s' should be installed.  This group does"
" not exist.  Would you like to ignore this group and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:195
#, python-format
msgid ""
"You have specified that the group '%s' should be excluded from installation."
"  This group does not exist.  Would you like to ignore this group and "
"continue with installation?"
msgstr ""

#: pyanaconda/errors.py:207
#, python-format
msgid ""
"The package '%s' is required for this installation. This package does not "
"exist. This is a fatal error and installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:214
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist.  Would you like to ignore this package and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:235
msgid ""
"Some packages, groups or modules are broken, the installation will be "
"aborted."
msgstr ""

#: pyanaconda/errors.py:240
msgid "Would you like to ignore this and continue with installation?"
msgstr ""

#: pyanaconda/errors.py:247
#, python-format
msgid ""
"Stream was not specified for a module without a default stream. This is a fatal error and installation will be aborted. The details of this error are:\n"
"\n"
"%(exception)s"
msgstr ""

#: pyanaconda/errors.py:255
#, python-format
msgid ""
"Multiple streams have been specified for a single module. This is a fatal error and installation will be aborted. The details of this error are:\n"
"\n"
"%(exception)s"
msgstr ""

#: pyanaconda/errors.py:263
#, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  This is a fatal error and installation will be aborted.  The details of this error are:\n"
"\n"
"%(details)s"
msgstr ""

#: pyanaconda/errors.py:271
msgid ""
"The following error occurred while installing.  This is a fatal error and "
"installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:279
msgid ""
"The following software marked for installation has errors.  This is likely caused by an error with\n"
"your installation source."
msgstr ""

#: pyanaconda/errors.py:287
msgid ""
"The following error occurred while installing the boot loader. The system "
"will not be bootable. Would you like to ignore this and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:299
#, python-format
msgid "Unable to encrypt password: unsupported algorithm %s"
msgstr ""

#: pyanaconda/errors.py:306
#, python-format
msgid ""
"Installation was stopped due to an error installing the boot loader. The exact error message is:\n"
"\n"
"%s\n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/errors.py:314
msgid ""
"Failed to connect the system to Red Hat Insights.\n"
"\n"
"Would you like to ignore this and continue with installation?"
msgstr ""

#: pyanaconda/errors.py:324
msgid ""
"The insights-client utility needed to connect the system to Red Hat Insights is not installed on the target system. If you want to connect the system to Red Hat Insights please make sure the insights-client package gets installed.\n"
"\n"
"Would you like to ignore this and continue with installation?"
msgstr ""

#: pyanaconda/storage/autopart.py:280
msgid "No big enough free space on disks for automatic partitioning"
msgstr ""

#: pyanaconda/storage/autopart.py:450 pyanaconda/storage/autopart.py:499
msgid "No usable disks selected"
msgstr ""

#: pyanaconda/storage/autopart.py:455 pyanaconda/storage/autopart.py:510
msgid "Not enough free space on disks for automatic partitioning"
msgstr ""

#: pyanaconda/storage/osinstall.py:347
#, python-format
msgid "Linux on %s"
msgstr ""

#: pyanaconda/storage/osinstall.py:352
msgid "Unknown Linux"
msgstr ""

#: pyanaconda/storage/osinstall.py:354
#, python-format
msgid "%(product)s %(version)s for %(arch)s"
msgstr ""

#: pyanaconda/storage/osinstall.py:357
#, python-format
msgid "%(product)s Linux %(version)s for %(arch)s"
msgstr ""

#: pyanaconda/storage/osinstall.py:2051
#: pyanaconda/ui/gui/spokes/custom_storage.py:727
msgid "You must create a new file system on the root device."
msgstr ""

#: pyanaconda/kickstart.py:219
#, python-format
msgid "Escrow certificate %s requires the network."
msgstr ""

#: pyanaconda/kickstart.py:227
#, python-format
msgid ""
"SSL error while downloading the escrow certificate:\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/kickstart.py:230
#, python-format
msgid ""
"The following error was encountered while downloading the escrow certificate:\n"
"\n"
"%s"
msgstr ""
"El fallu siguiente alcontróse mentantu se descargaba'l certificáu escrow:\n"
"\n"
"%s"

#: pyanaconda/kickstart.py:380
#, python-format
msgid "%s is missing. Cannot setup authentication."
msgstr ""

#: pyanaconda/kickstart.py:462
msgid "GRUB2 does not support installation to a partition."
msgstr ""

#: pyanaconda/kickstart.py:469
msgid "GRUB2 encrypted password must be in grub.pbkdf2 format."
msgstr ""

#: pyanaconda/kickstart.py:618
msgid "No match found for given boot drive \"{}\"."
msgstr ""

#: pyanaconda/kickstart.py:622
msgid "More than one match found for given boot drive \"{}\"."
msgstr ""

#: pyanaconda/kickstart.py:626
msgid "Requested boot drive \"{}\" doesn't exist or cannot be used."
msgstr ""

#: pyanaconda/kickstart.py:707
msgid "Btrfs file system is not supported."
msgstr ""

#: pyanaconda/kickstart.py:809
#, python-format
msgid "NIC \"%s\" given in fcoe command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:879
#, python-format
msgid "%s is missing. Cannot setup firewall."
msgstr ""

#: pyanaconda/kickstart.py:935
#, python-format
msgid ""
"Network interface \"%(nic)s\" required by iSCSI \"%(iscsiTarget)s\" target "
"is not up."
msgstr ""

#: pyanaconda/kickstart.py:945
msgid ""
"iscsi --iface must be specified (binding used) either for all targets or for"
" none"
msgstr ""

#: pyanaconda/kickstart.py:1050 pyanaconda/kickstart.py:1784
#, python-format
msgid "The mount point \"%s\" is not valid.  It must start with a /."
msgstr ""

#: pyanaconda/kickstart.py:1056
#, python-format
msgid ""
"No volume group exists with the name \"%s\".  Specify volume groups before "
"logical volumes."
msgstr ""

#: pyanaconda/kickstart.py:1063
msgid "Cache PVs must belong to the same VG as the cached LV"
msgstr ""

#: pyanaconda/kickstart.py:1069
#, python-format
msgid ""
"No thin pool exists with the name \"%s\". Specify thin pools before thin "
"volumes."
msgstr ""

#: pyanaconda/kickstart.py:1077
msgid "logvol --noformat must also use the --name= option."
msgstr ""

#: pyanaconda/kickstart.py:1082 pyanaconda/kickstart.py:1144
#, python-format
msgid "Logical volume \"%s\" given in logvol command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1093 pyanaconda/kickstart.py:1102
#: pyanaconda/kickstart.py:1154 pyanaconda/kickstart.py:1546
#: pyanaconda/kickstart.py:1555 pyanaconda/kickstart.py:1638
#, python-format
msgid "Target size \"%(size)s\" for device \"%(device)s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:1116
#, python-format
msgid ""
"Logical volume name \"%(logvol)s\" is already in use in volume group "
"\"%(volgroup)s\"."
msgstr ""

#: pyanaconda/kickstart.py:1121
#, python-format
msgid ""
"Logical volume size \"%(logvolSize)s\" must be larger than the volume group "
"extent size of \"%(extentSize)s\"."
msgstr ""

#: pyanaconda/kickstart.py:1133 pyanaconda/kickstart.py:1574
#: pyanaconda/kickstart.py:1837
#, python-format
msgid "The \"%s\" file system type is not supported."
msgstr ""

#: pyanaconda/kickstart.py:1328
#, python-format
msgid "Unknown or invalid device '%s' specified"
msgstr ""

#: pyanaconda/kickstart.py:1334
#, python-format
msgid ""
"Unknown or invalid format '%(format)s' specified for device '%(device)s'"
msgstr ""

#: pyanaconda/kickstart.py:1341
#, python-format
msgid "No format on device '%s'"
msgstr ""

#: pyanaconda/kickstart.py:1393 pyanaconda/kickstart.py:1401
#, python-format
msgid "nvdimm: namespace %s not found."
msgstr ""

#: pyanaconda/kickstart.py:1412
#, python-format
msgid "Disk \"%s\" given in nvdimm command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1443
#, python-format
msgid "No disk found for specified BIOS disk \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:1478
#, python-format
msgid "RAID partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1489 pyanaconda/kickstart.py:1760
#, python-format
msgid "PV partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1500 pyanaconda/kickstart.py:1770
#, python-format
msgid "Btrfs partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1523
#, python-format
msgid "The size \"%s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:1530
msgid "part --noformat must also use the --onpart option."
msgstr ""

#: pyanaconda/kickstart.py:1535 pyanaconda/kickstart.py:1629
#, python-format
msgid "Partition \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1589 pyanaconda/kickstart.py:1603
#, python-format
msgid "Disk \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1592
#, python-format
msgid "Cannot install to unpartitionable device \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:1599
#, python-format
msgid "Disk \"%s\" in part command is not partitioned."
msgstr ""

#: pyanaconda/kickstart.py:1612
#, python-format
msgid "The maximum size \"%s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:1791
msgid "raid --noformat must also use the --device option."
msgstr ""

#: pyanaconda/kickstart.py:1796
#, python-format
msgid "RAID device  \"%s\" given in raid command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1819
#, python-format
msgid ""
"RAID device \"%(device)s\" has a format of \"%(format)s\", but should have a"
" format of \"mdmember\"."
msgstr ""

#: pyanaconda/kickstart.py:1824
#, python-format
msgid "Tried to use undefined partition \"%s\" in RAID specification."
msgstr ""

#: pyanaconda/kickstart.py:1858
#, python-format
msgid "RAID volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1867
#, python-format
msgid "The RAID volume name \"%s\" is already in use."
msgstr ""

#: pyanaconda/kickstart.py:2230
#, python-format
msgid ""
"Physical volume \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"lvmpv\"."
msgstr ""

#: pyanaconda/kickstart.py:2235
#, python-format
msgid "Tried to use undefined partition \"%s\" in Volume Group specification"
msgstr ""

#: pyanaconda/kickstart.py:2241
#, python-format
msgid ""
"Volume group \"%s\" defined without any physical volumes.  Either specify "
"physical volumes or use --useexisting."
msgstr ""

#: pyanaconda/kickstart.py:2251
#, python-format
msgid ""
"Volume group given physical extent size of \"%(extentSize)s\", but must be one of:\n"
"%(validExtentSizes)s."
msgstr ""

#: pyanaconda/kickstart.py:2258
msgid ""
"volgroup --noformat and volgroup --useexisting must also use the --name= "
"option."
msgstr ""

#: pyanaconda/kickstart.py:2263
#, python-format
msgid "Volume group \"%s\" given in volgroup command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:2266
#, python-format
msgid "The volume group name \"%s\" is already in use."
msgstr ""

#: pyanaconda/kickstart.py:2397
msgid ""
"Incorrectly specified origin of the snapshot. Use format \"VolGroup/LV-"
"name\""
msgstr ""

#: pyanaconda/kickstart.py:2407
#, python-format
msgid "Snapshot: origin \"%s\" doesn't exists!"
msgstr ""

#: pyanaconda/kickstart.py:2411
#, python-format
msgid ""
"Snapshot: origin \"%(origin)s\" of snapshot \"%(name)s\" is not a valid thin"
" LV device."
msgstr ""

#: pyanaconda/kickstart.py:2417
#, python-format
msgid "Snapshot %s already exists."
msgstr ""

#: pyanaconda/kickstart.py:2766
msgid ""
"\n"
"Some warnings occurred during reading the kickstart file:"
msgstr ""

#: pyanaconda/kickstart.py:2771
#, python-format
msgid ""
"\n"
"An error occurred during reading the kickstart file:\n"
"%s\n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/rescue.py:316
msgid "Unable to find /bin/bash to execute!  Not starting shell."
msgstr ""

#: pyanaconda/rescue.py:333
msgid "Rescue"
msgstr "Rescatar"

#: pyanaconda/rescue.py:340
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and mount it under the directory : %s.  You can then make any changes required to your system.  Choose '1' to proceed with this step.\n"
"You can choose to mount your file systems read-only instead of read-write by choosing '2'.\n"
"If for some reason this process does not work choose '3' to skip directly to a shell.\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:352
msgid "Continue"
msgstr "Siguir"

#: pyanaconda/rescue.py:353
msgid "Read-only mount"
msgstr ""

#: pyanaconda/rescue.py:354
msgid "Skip to shell"
msgstr ""

#: pyanaconda/rescue.py:355
msgid "Quit (Reboot)"
msgstr ""

#: pyanaconda/rescue.py:370 pyanaconda/ui/tui/spokes/askvnc.py:104
msgid "Do you really want to quit?"
msgstr ""

#: pyanaconda/rescue.py:461
msgid "Rescue Shell"
msgstr ""

#: pyanaconda/rescue.py:471
#, python-format
msgid "Run %s to unmount the system when you are finished."
msgstr ""

#: pyanaconda/rescue.py:472
msgid "When finished, please exit from the shell and your system will reboot.\n"
msgstr ""

#: pyanaconda/rescue.py:483
#, python-format
msgid ""
"Your system has been mounted under %(mountpoint)s.\n"
"\n"
"If you would like to make the root of your system the root of the active system, run the command:\n"
"\n"
"\tchroot %(mountpoint)s\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:493
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system.  Some of it may be mounted under %s\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:498
msgid "Rebooting."
msgstr ""

#: pyanaconda/rescue.py:501
#, python-format
msgid "You don't have any Linux partitions. %s\n"
msgstr ""

#: pyanaconda/rescue.py:507
msgid "Not mounting the system.\n"
msgstr ""

#: pyanaconda/rescue.py:521
#, python-format
msgid "Please press %s to get a shell"
msgstr ""

#: pyanaconda/rescue.py:538
msgid "Root Selection"
msgstr ""

#: pyanaconda/rescue.py:564
msgid "The following installations were discovered on your system."
msgstr ""

#: pyanaconda/subscription.py:154
msgid "Registration faile due to insufficient credentials."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:60
msgid "File system type \"{}\" given in autopart command is invalid."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:82
msgid ""
"Disklabel \"{}\" given in clearpart command is not supported on this "
"platform."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:88
msgid "Disk \"{}\" given in clearpart command does not exist."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:93
msgid "Device \"{}\" given in clearpart device list does not exist."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:112
#: pyanaconda/modules/storage/kickstart.py:117
msgid "Disk \"{}\" given in ignoredisk command does not exist."
msgstr ""

#: pyanaconda/modules/subscription/installation.py:207
#: pyanaconda/modules/subscription/installation.py:255
#: pyanaconda/ui/gui/spokes/subscription.py:305
msgid "Registration failed."
msgstr ""

#: pyanaconda/modules/subscription/installation.py:284
msgid "Unregistration failed."
msgstr ""

#: pyanaconda/modules/subscription/installation.py:321
msgid "Failed to attach subscription."
msgstr ""

#: pyanaconda/modules/subscription/subscription_data.py:197
msgid "product name unknown"
msgstr ""

#: pyanaconda/modules/subscription/subscription_data.py:201
#: pyanaconda/modules/subscription/subscription_data.py:205
#: pyanaconda/modules/subscription/subscription_data.py:212
#: pyanaconda/modules/subscription/subscription_data.py:216
#: pyanaconda/modules/subscription/subscription_data.py:225
msgid "unknown"
msgstr ""

#: pyanaconda/modules/subscription/subscription_data.py:208
msgid "Not Available"
msgstr ""

#: pyanaconda/startup_utils.py:175
#, python-format
msgid ""
"%(product_name)s requires %(needed_ram)s MB of memory to install, but you "
"only have %(total_ram)s MB on this machine.\n"
msgstr ""

#: pyanaconda/startup_utils.py:177
#, python-format
msgid ""
"The %(product_name)s graphical installer requires %(needed_ram)s MB of memory, but you only have %(total_ram)s MB\n"
"."
msgstr ""

#: pyanaconda/startup_utils.py:180
msgid ""
"\n"
"Press [Enter] to reboot your system.\n"
msgstr ""

#: pyanaconda/startup_utils.py:182
msgid "Not enough RAM"
msgstr "Nun hai RAM bastante"

#: pyanaconda/startup_utils.py:183
msgid ""
" Try the text mode installer by running:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" from a root terminal."
msgstr ""

#: pyanaconda/startup_utils.py:185
msgid " Starting text mode."
msgstr "Aniciar en mou testu"

#: pyanaconda/startup_utils.py:222
msgid "The installation cannot continue and the system will be rebooted"
msgstr ""

#: pyanaconda/startup_utils.py:223
msgid "Press ENTER to continue"
msgstr ""

#: pyanaconda/startup_utils.py:320
#, python-format
msgid "Please ssh install@%s to begin the install."
msgstr ""

#: pyanaconda/startup_utils.py:322
msgid "Please ssh install@HOSTNAME to continue installation."
msgstr ""

#: pyanaconda/users.py:56
#, python-format
msgid "User name is reserved for system: %s"
msgstr ""

#: pyanaconda/users.py:73
msgid ""
"Either a group name in the group list is invalid or groups are not separated"
" by a comma."
msgstr ""

#: pyanaconda/users.py:84
msgid "Name cannot start with '-' character."
msgstr ""

#: pyanaconda/users.py:87
#, python-format
msgid "Name '%s' is not allowed."
msgstr ""

#: pyanaconda/users.py:90
msgid "Fully numeric name is not allowed."
msgstr ""

#: pyanaconda/users.py:94
msgid "Name '$' is not allowed."
msgstr ""

#: pyanaconda/users.py:104
#, python-format
msgid "Name cannot contain character: '%s'"
msgstr ""

#: pyanaconda/users.py:107
msgid "Name must be shorter than 33 characters."
msgstr ""

#: pyanaconda/users.py:111
#, python-format
msgid "Name '%s' is invalid."
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:46
msgid "Installation"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:60
msgid "Starting automated install"
msgstr ""

#. TRANSLATORS: 'b' to begin installation
#: pyanaconda/ui/tui/hubs/summary.py:105
#, python-format
msgid "Enter '%s' to ignore the warning and attempt to install anyway."
msgstr ""

#. TRANSLATORS: 'b' to begin installation
#: pyanaconda/ui/tui/hubs/summary.py:107 pyanaconda/ui/tui/hubs/summary.py:134
#: pyanaconda/ui/tui/hubs/summary.py:146
msgctxt "TUI|Spoke Navigation"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:116
msgid "The following mandatory spokes are not completed:"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:134
msgid "to begin installation"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:149
#: pyanaconda/ui/tui/hubs/__init__.py:137
msgid "Please complete all spokes before continuing"
msgstr ""

#. TRANSLATORS: 'c' to continue
#: pyanaconda/ui/tui/hubs/summary.py:159
#: pyanaconda/ui/tui/spokes/installation_source.py:499
#: pyanaconda/ui/tui/spokes/software_selection.py:252
#: pyanaconda/ui/tui/spokes/storage.py:290
#: pyanaconda/ui/tui/spokes/storage.py:647
#: pyanaconda/ui/tui/spokes/storage.py:714
#: pyanaconda/ui/tui/spokes/storage.py:781
msgctxt "TUI|Spoke Navigation"
msgid "c"
msgstr ""

#: pyanaconda/ui/tui/hubs/__init__.py:54
msgid "Default HUB title"
msgstr ""

#: pyanaconda/ui/tui/hubs/__init__.py:162
#, python-format
msgid "to enter the %(spoke_title)s spoke"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:144
#, python-format
msgid "Enter a new value for '%(title)s' and press %(enter)s"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:273
msgid ""
"\n"
"Would you like to use it anyway?"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:66
msgid "Installation source"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:111
#: pyanaconda/ui/gui/spokes/installation_source.py:707
#, python-format
msgid "NFS server %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:113
#: pyanaconda/ui/gui/spokes/installation_source.py:709
msgid "Local media"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:115
#: pyanaconda/ui/gui/spokes/installation_source.py:711
msgid "Local media via SE/HMC"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:118
#: pyanaconda/ui/tui/spokes/installation_source.py:132
#: pyanaconda/ui/gui/spokes/installation_source.py:703
msgid "Error setting up software source"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:121
#: pyanaconda/ui/tui/spokes/installation_source.py:160
#: pyanaconda/ui/gui/spokes/installation_source.py:720
#: pyanaconda/ui/gui/spokes/installation_source.glade:800
msgid "Closest mirror"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:123
#: pyanaconda/ui/tui/spokes/software_selection.py:171
#: pyanaconda/ui/gui/spokes/installation_source.py:722
#: pyanaconda/ui/gui/spokes/software_selection.py:329
#: pyanaconda/ui/gui/spokes/software_selection.py:334
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:578
msgid "Nothing selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:134
#: pyanaconda/ui/tui/spokes/software_selection.py:159
msgid "Processing..."
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:154
#: pyanaconda/ui/gui/spokes/installation_source.py:1109
msgid ""
"The installation source is in use by the installer and cannot be changed."
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:166
msgid "Choose an installation source type."
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:167
msgid "CD/DVD"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:168
msgid "local ISO file"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:169
msgid "Network"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:172
msgid "SE/HMC"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:262
#: pyanaconda/ui/tui/spokes/installation_source.py:315
msgid "Specify Repo Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:274
msgid "Repo URL"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:334
msgid "SERVER:/PATH"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:337
msgid "NFS mount options"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:347
msgid "Server must be specified as SERVER:/PATH"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:391
msgid "Select device containing the ISO file"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:438
msgid "No mountable devices found"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:469
msgid "Select an ISO to use as install source"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:487
msgid "No *.iso files found in device root folder"
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:41
msgid "Shell"
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:55
msgid "Start shell"
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:63
msgid "Exit the shell to continue"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:38
msgid "Full name can't contain the ':' character"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:67
msgid "User creation"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:96
msgid "Create user"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:100
msgid "Full name"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:103
#: pyanaconda/ui/gui/spokes/subscription.glade:156
#: pyanaconda/ui/gui/spokes/subscription.glade:552
msgid "User name"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:106
msgid "Use password"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:110
#: pyanaconda/ui/tui/spokes/root_password.py:96
#: pyanaconda/ui/gui/spokes/root_password.glade:87
#: pyanaconda/ui/gui/spokes/user.glade:172
#: pyanaconda/ui/gui/spokes/subscription.glade:181
#: pyanaconda/ui/gui/spokes/subscription.glade:564
msgid "Password"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:118
msgid "Administrator"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:122
msgid "Groups"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:197 pyanaconda/ui/gui/spokes/user.py:412
msgid "No user will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:199
msgid "You must set a password"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:201 pyanaconda/ui/gui/spokes/user.py:414
#, python-format
msgid "Administrator %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:203 pyanaconda/ui/gui/spokes/user.py:416
#, python-format
msgid "User %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:217
#, python-format
msgid ""
"Invalid user name: %(name)s.\n"
"%(error_message)s"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:44
msgid "Bad format of the IP address"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:45
msgid "Bad format of the netmask"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:61
msgid "Network configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:127
#, python-format
msgid "Wired (%(interface_name)s) disconnected\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:132
#, python-format
msgid "Wired (%(interface_name)s) connected\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:145
#, python-format
msgid " IPv4 Address: %(addr)s Netmask: %(netmask)s Gateway: %(gateway)s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:147
#, python-format
msgid " DNS: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:154
#, python-format
msgid " IPv6 Address: %(addr)s/%(prefix)d\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:168
#, python-format
msgid "Host Name: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:170
#, python-format
msgid "Current host name: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:177
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:239
#: pyanaconda/ui/gui/spokes/network.glade:2200
msgid "Host Name"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:178
msgid "Set host name"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:181
#, python-format
msgid "Configure device %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:222
msgid "Configuration not saved: netmask missing in static configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:240
msgid "Can't apply configuration, device activation failed."
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:286
#: pyanaconda/ui/gui/spokes/network.py:1646
#: pyanaconda/ui/gui/spokes/network.py:1664
#: pyanaconda/ui/gui/spokes/network.py:1754
#: pyanaconda/ui/gui/spokes/network.py:1772
#, python-format
msgid "Host name is not valid: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:297
msgid "Device configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:314
#, python-format
msgid "IPv4 address or %s for DHCP"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:318
msgid "IPv4 netmask"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:321
msgid "IPv4 gateway"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:324
#, python-format
msgid ""
"IPv6 address[/prefix] or %(auto)s for automatic, %(dhcp)s for DHCP, "
"%(ignore)s to turn off"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:330
msgid "IPv6 default gateway"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:333
msgid "Nameservers (comma separated)"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:336
msgid "Connect automatically after reboot"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:340
msgid "Apply configuration in installer"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:346
#, python-format
msgid "Configuring device %s."
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:57
msgid "Default spoke title"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:61
msgid "testing status..."
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:49
msgid "VNC"
msgstr ""

#. TRANSLATORS: 'q' to quit
#: pyanaconda/ui/tui/spokes/askvnc.py:103
msgctxt "TUI|Spoke Navigation"
msgid "q"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:127
msgid "VNC Password"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:132
msgid ""
"Please provide VNC password (must be six to eight characters long).\n"
"You will have to type it twice. Leave blank for no password"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:149
#: pyanaconda/ui/gui/spokes/network.py:1471
msgid "Password: "
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:150
msgid "Password (confirm): "
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:153
msgid "Passwords do not match!"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:155
msgid "The password must be at least six characters long."
msgstr "La contraseña tien de tener al menos 6 carauteres de llargu."

#: pyanaconda/ui/tui/spokes/askvnc.py:158
msgid "The password cannot be more than eight characters long."
msgstr ""

#: pyanaconda/ui/tui/spokes/root_password.py:43
msgid "Root password"
msgstr ""

#: pyanaconda/ui/tui/spokes/root_password.py:78
msgid "Disabled. Set password to enable root account."
msgstr ""

#: pyanaconda/ui/tui/spokes/root_password.py:80
#: pyanaconda/ui/gui/spokes/root_password.py:158
msgid "Root account is disabled."
msgstr ""

#: pyanaconda/ui/tui/spokes/root_password.py:83
msgid "Password is set."
msgstr ""

#: pyanaconda/ui/tui/spokes/root_password.py:85
msgid "Password is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/root_password.py:90
msgid "Please select new root password. You will have to type it twice."
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_progress.py:50
msgid "Progress"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_progress.py:136
msgid "Installation complete"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_progress.py:138
msgid "Use of this product is subject to the license agreement found at:"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_progress.py:149
#, python-format
msgid "Installation complete. Press %s to quit"
msgstr ""

#: pyanaconda/ui/tui/spokes/software_selection.py:52
msgid "Software selection"
msgstr ""

#: pyanaconda/ui/tui/spokes/software_selection.py:157
#: pyanaconda/ui/gui/spokes/software_selection.py:304
msgid "Error checking software selection"
msgstr ""

#: pyanaconda/ui/tui/spokes/software_selection.py:161
#: pyanaconda/ui/gui/spokes/software_selection.py:313
msgid "Installation source not set up"
msgstr ""

#: pyanaconda/ui/tui/spokes/software_selection.py:163
#: pyanaconda/ui/gui/spokes/software_selection.py:316
msgid "Source changed - please verify"
msgstr ""

#: pyanaconda/ui/tui/spokes/software_selection.py:170
#: pyanaconda/ui/gui/spokes/software_selection.py:323
msgid "Custom software selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/software_selection.py:198
msgid "Installation source needs to be set up first."
msgstr ""

#: pyanaconda/ui/tui/spokes/software_selection.py:222
msgid "Base environment"
msgstr ""

#: pyanaconda/ui/tui/spokes/software_selection.py:232
msgid "Additional software for selected environment"
msgstr ""

#: pyanaconda/ui/tui/spokes/software_selection.py:234
msgid "No additional software to select."
msgstr ""

#: pyanaconda/ui/tui/spokes/unsupported_hardware.py:38
#: pyanaconda/ui/gui/spokes/lib/unsupported_hardware.glade:68
msgid "Unsupported Hardware Detected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:66
msgid "Use All Space"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:67
msgid "Replace Existing Linux system(s)"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:68
msgid "Use Free Space"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:101
msgid "Installation Destination"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:142
#: pyanaconda/ui/gui/spokes/storage.py:573
msgid "Kickstart insufficient"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:144
#: pyanaconda/ui/gui/spokes/storage.py:485
#: pyanaconda/ui/gui/spokes/storage.py:575
msgid "No disks selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:146
#: pyanaconda/ui/gui/spokes/storage.py:577
msgid "Error checking storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:148
#: pyanaconda/ui/gui/spokes/storage.py:579
msgid "Warning checking storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:150
#: pyanaconda/ui/gui/spokes/storage.py:581
msgid "Automatic partitioning selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:152
#: pyanaconda/ui/gui/spokes/storage.py:583
msgid "Custom partitioning selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:186
#: pyanaconda/ui/gui/spokes/storage.py:874
msgid ""
"No disks detected.  Please shut down the computer, connect at least one "
"disk, and restart to complete installation."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:189
#: pyanaconda/ui/gui/spokes/storage.py:878
msgid "No disks selected; please select at least one disk to install to."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:231
msgid "Select all"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:337
msgid ""
"The following unformatted or LDL DASDs have been detected on your system. You can choose to format them now with dasdfmt or cancel to leave them unformatted. Unformatted DASDs cannot be used during installation.\n"
"\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:344
msgid ""
"Warning: All storage changes made using the installer will be lost when you choose to format.\n"
"\n"
"Proceed to run dasdfmt?\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:353
msgid "This may take a moment."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:372
msgid "Passphrase"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:373
msgid ""
"Please provide a default LUKS passphrase for all devices you want to "
"encrypt. You will have to type it twice."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:449
msgid "Generating updated storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:454
#: pyanaconda/ui/tui/spokes/storage.py:470
#, python-format
msgid "storage configuration failed: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:474 pyanaconda/ui/helpers.py:94
msgid "Checking storage configuration..."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:542
msgid "Partitioning Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:575
msgid "Manually assign mount points"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:581
msgid ""
"Installation requires partitioning of your hard drive. Select what space to "
"use for the install target or manually assign mount points."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:631
msgid "Reverting previous configuration. This may take a moment..."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:670
msgid "Partition Scheme Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:704
msgid "Select a partition scheme configuration."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:733
msgid "Assign mount points"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:754
msgid ""
"Choose device from above to assign mount point and set format.\n"
"Formats marked with * are new formats meaning ALL DATA on the original format WILL BE LOST!"
msgstr ""

#. TRANSLATORS: 's' to rescan devices
#: pyanaconda/ui/tui/spokes/storage.py:766
#: pyanaconda/ui/tui/spokes/storage.py:776
msgctxt "TUI|Spoke Navigation|Partitioning"
msgid "s"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:766
msgid "rescan devices"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:872
msgid ""
"Warning: This will revert all changes done so far.\n"
"Do you want to proceed?\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:887
msgid "Scanning disks. This may take a moment..."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:901
#, python-format
msgid "Configure device: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:922
msgid "Choose from above to assign mount point and/or set format."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:938
#: pyanaconda/ui/gui/spokes/lib/summary.glade:177
msgid "Mount point"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:943
#: pyanaconda/ui/tui/spokes/storage.py:948
#: pyanaconda/ui/tui/spokes/storage.py:980
msgid "none"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:966
msgid "Invalid mount point given"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:979
msgid "Format"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:989
msgid "Invalid or unsupported format given"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:991
#, python-format
msgid "Supported formats: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:1008
msgid "Reformat"
msgstr ""

#: pyanaconda/ui/tui/spokes/kernel_warning.py:36
msgid "Warning: Processor has Simultaneous Multithreading (SMT) enabled"
msgstr ""

#: pyanaconda/ui/tui/spokes/language_support.py:51
msgid "Language settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/language_support.py:86
msgid "Language is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/language_support.py:98
msgid "Available locales"
msgstr ""

#: pyanaconda/ui/tui/spokes/language_support.py:103
msgid "Available languages"
msgstr ""

#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/spokes/language_support.py:127
#: pyanaconda/ui/tui/spokes/language_support.py:138
msgctxt "TUI|Spoke Navigation|Language Support"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/spokes/language_support.py:136
msgid "Please select language support to install"
msgstr ""

#: pyanaconda/ui/tui/spokes/language_support.py:138
msgid "to return to language list"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:47
msgid "status: working"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:48
msgid "status: not working"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:49
msgid "checking status"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:65
msgid "Time settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:203
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:570
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:576
#, python-format
msgid "%s timezone"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:205
msgid "Timezone is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:216
msgid "not set"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:220
#, python-format
msgid "Timezone: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:226
#: pyanaconda/ui/tui/spokes/time_spoke.py:393
msgid "NTP servers:"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:231
msgid "not configured"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:242
msgid "Change timezone"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:244
msgid "Set timezone"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:249
msgid "Configure NTP servers"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:286
msgid "Timezone settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:314
#, python-format
msgid "Available timezones in region %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:318
msgid "Available regions"
msgstr ""

#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/spokes/time_spoke.py:359
#: pyanaconda/ui/tui/spokes/time_spoke.py:370
msgctxt "TUI|Spoke Navigation|Time Settings"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:368
msgid "Please select the timezone. Use numbers or type names directly"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:370
msgid "back to region list"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:383
msgid "NTP configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:398
msgid "no NTP servers have been configured"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:409
msgid "Add NTP server"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:413
msgid "Remove NTP server"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:444
msgid "Add NTP server address"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:460
#, python-format
msgid "Enter an NTP server address and press %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:484
msgid "Select an NTP server to remove"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:118
msgid "Complete!"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:147
#, python-format
msgid "Use of this product is subject to the license agreement found at %s"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:222
#, python-format
msgid ""
"%s is now successfully installed on your system and ready for you to use!  "
"When you are ready, reboot your system to start using it!"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:225
msgctxt "GUI|Progress"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:233
#, python-format
msgid ""
"%s is now successfully installed, but some configuration still needs to be done.\n"
"Finish it and then click the Finish configuration button please."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:238
#, python-format
msgid ""
"%s is now successfully installed and ready for you to use!\n"
"Go ahead and reboot to start using it!"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.py:92
msgid ""
"Warning: Processor has Simultaneous Multithreading (SMT) enabled.  <a "
"href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.py:102
msgid ""
"The following warnings were encountered when checking your kernel "
"configuration. These are not fatal, but you may wish to make changes to your"
" kernel config."
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.py:110
msgctxt "GUI|Summary|Warning Dialog"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/hubs/__init__.py:274
msgid ""
"Please complete items marked with this icon before continuing to the next "
"step."
msgstr ""

#: pyanaconda/ui/gui/__init__.py:410 widgets/src/BaseWindow.c:151
msgid "Help!"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:905
#: pyanaconda/ui/gui/spokes/lib/detailederror.py:57
msgctxt "GUI|Detailed Error Dialog"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:921
msgctxt "GUI|Yes No Dialog"
msgid "_No"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:922
msgctxt "GUI|Yes No Dialog"
msgid "_Yes"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:71
msgctxt "GUI|Spoke"
msgid "_Connect to Red Hat"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:297
msgid "Unregistering..."
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:299
msgid "Registering..."
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:301
msgid "Attaching subscription..."
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:303
msgid "Subscription attached."
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:307
msgid "Registered."
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:309
msgid "Not registered."
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:397
msgid "Not Specified"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:437
msgid "Other ({})"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:635
#: pyanaconda/core/constants.py:219
msgid "Password set."
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:725
msgid "Activation key set."
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:837
msgid "Please enable network access before connecting to Red Hat."
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:886
msgid "Registered with account {}"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:888
msgid "Registered with organization {}"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:891
msgid "Role: {}"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:894
msgid "SLA: {}"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:904
msgid "Connected to Red Hat Insights"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:906
msgid "Not connected to Red Hat Insights"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:918
msgid "No subscriptions are attached to the system"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:920
msgid "1 subscription attached to the system"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:922
msgid "{} subscriptions attached to the system"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:988
msgid "Service level"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:991
msgid "SKU"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:994
msgid "Contract"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:1007
msgid "Start date"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:1010
msgid "End date"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:1013
msgid "Entitlements"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.py:1015
msgid "{} consumed"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:59
msgid "Setting up installation source..."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:89
msgid " <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:100
msgid "Invalid proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:105
#, python-format
msgid "Invalid proxy protocol: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:111
msgid "Extra characters in proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:115
msgid "Proxy authentication data duplicated"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:251
msgid "This media is good to install from."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:253
msgid "This media is not good to install from."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:387
msgctxt "GUI|Spoke"
msgid "_Installation Source"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:569
msgid "Failed to set up installation source; check the repo url"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:688
#: pyanaconda/ui/gui/spokes/installation_source.py:699
msgid "Red Hat CDN"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:690
#: pyanaconda/ui/gui/spokes/software_selection.py:226
msgid "Checking software dependencies..."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:701
msgid "Error setting up base repository"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:717
msgid "Error setting up installation from HDD"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:879
msgid "Failed to set up installation source; check the repo url."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:881
msgid ""
"Failed to set up installation source; check the repo url and proxy settings."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:946
#, python-format
msgid "Device: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:947
#, python-format
msgid "Label: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1116
msgid ""
"You need to configure the network to use a network installation source."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1214
#, python-format
msgid "Repository %s has empty url"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1216
msgid "URL is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1221
#, python-format
msgid "Repository %s has invalid url"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1223
msgid "Invalid URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1230
#, python-format
msgid "Repository %s does not match selected protocol"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1232
msgid "Protocol in URL does not match selected protocol"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1236
#, python-format
msgid "Repository %s has empty NFS server"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1238
msgid "NFS server is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1246
#, python-format
msgid "Repository %s has invalid host name"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1248
msgid "Invalid host name"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1252
#, python-format
msgid "Repository %s required remote directory"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1254
msgid "Remote directory is required"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1275
msgid "Duplicate repository names."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1283
msgid "Empty repository name"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1286
msgid "Invalid repository name"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1291
msgid "Repository name conflicts with internal repository name."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.py:240
#, python-format
msgid "WELCOME TO %(name)s %(version)s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:114
#, python-format
msgid "Create a new %(container_type)s ..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:115
#, python-format
msgid "Create or select %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:117
msgid "Device reconfiguration failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:119
msgid ""
"Storage configuration reset due to unrecoverable error. <a href=\"\">Click "
"for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:157
#: pyanaconda/ui/gui/spokes/custom_storage.glade:105
msgid "MANUAL PARTITIONING"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:388
#, python-format
msgctxt "GUI|Custom Partitioning"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:483
#, python-format
msgid "%(items_selected)s of %(items_total)s mount points in %(page_name)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:509
#, python-format
msgid ""
"Please select a single mount point to edit properties.\n"
"\n"
"You have currently selected:\n"
"%s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:513
#, python-format
msgid ""
"When you create mount points for your %(name)s %(version)s installation, "
"you'll be able to view their details here."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:599
#: pyanaconda/ui/gui/spokes/network.py:275
#: pyanaconda/ui/gui/spokes/lib/accordion.py:65 pyanaconda/network.py:1670
msgid "Unknown"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:716
#, python-format
msgid ""
"/boot/efi must be on a device of type %(oneFsType)s or %(anotherFsType)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:720
#, python-format
msgid "%(fs)s must be on a device of type %(type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:723
#: pyanaconda/ui/gui/spokes/custom_storage.py:725
#, python-format
msgid "%s cannot be encrypted"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:732
#, python-format
msgid "Device does not support RAID level selection %s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:845
msgid "Device resize request failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:928
msgid "Device reformat request failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1310
#: pyanaconda/ui/gui/spokes/custom_storage.py:2302
#, python-format
msgid "Invalid device name: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1316
#, python-format
msgid "Specified name %s already in use."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1552
msgid "No disks assigned"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1558
#, python-format
msgctxt "GUI|Custom Partitioning|Devices"
msgid " and %d other"
msgid_plural " and %d others"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1617
msgid "The container is encrypted."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1659
msgid ""
"The space available to this mount point can be changed by modifying the "
"volume below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1661
msgid "This file system may not be resized."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1717
#: pyanaconda/ui/gui/spokes/blivet_gui.py:202
msgid ""
"Error checking storage configuration.  <a href=\"\">Click for details</a> or"
" press Done again to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1719
#: pyanaconda/ui/gui/spokes/blivet_gui.py:204
msgid ""
"Warning checking storage configuration.  <a href=\"\">Click for details</a> "
"or press Done again to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1823
#, python-format
msgid "Added new %(type)s to existing container %(name)s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1834
msgid "Failed to add new device. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1839
msgid "Invalid partition size set. Use a valid integer."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1948
msgid "Device removal request failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2033
#, python-format
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "Delete _all file systems which are only used by %s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2037
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "Do _not show this dialog for other selected file systems."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2169
msgid "No disks selected. Keeping previous disk set."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2229
msgid "No disks selected. Not saving changes."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2237
#, python-format
msgid "Volume Group name %s is already in use. Not saving changes."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2269
#, python-format
msgid "(%s free)"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2439
#, python-format
msgid ""
"This Software RAID array is missing %(missingMembers)d of %(totalMembers)d "
"member partitions. You can remove it or select a different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2445
#, python-format
msgid ""
"This LVM Volume Group is missing %(missingPVs)d of %(totalPVs)d physical "
"volumes. You can remove it or select a different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2449
#, python-format
msgid ""
"This %(type)s device is missing member devices. You can remove it or select "
"a different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2532
msgid "No disks selected."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2537
msgid "Not enough free space on selected disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2542
#: pyanaconda/ui/gui/spokes/custom_storage.py:2548
#: pyanaconda/ui/gui/spokes/custom_storage.py:2573
msgid "Automatic partitioning failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2814
#: pyanaconda/ui/gui/spokes/blivet_gui.py:280
msgid ""
"Continuing with this action will reset all your partitioning selections to "
"their current on-disk state."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2822
#: pyanaconda/ui/gui/spokes/blivet_gui.py:288
msgctxt "GUI|Custom Partitioning|Reset Dialog"
msgid "_Reset selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2823
#: pyanaconda/ui/gui/spokes/blivet_gui.py:289
msgctxt "GUI|Custom Partitioning|Reset Dialog"
msgid "_Preserve current selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:2898
msgid ""
"Failed to unlock encrypted block device. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.py:276
#, python-format
msgid ""
"The following nodes were discovered using the iSCSI initiator "
"<b>%(initiatorName)s</b> using the target IP address "
"<b>%(targetAddress)s</b>.  Please select which nodes you wish to log into:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.py:78
msgctxt "GUI|Advanced Storage|NVDIM"
msgid "No device to be reconfigured selected."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.py:153
msgid "Rescanning disks finished."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.py:577
msgctxt "GUI|Spoke"
msgid "_Installation Destination"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.py:700
#, python-format
msgctxt "GUI|Installation Destination|Filter"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/user.py:66
msgid "Invalid group name."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:224
msgctxt "GUI|Spoke"
msgid "_User Creation"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:342
#: pyanaconda/ui/gui/spokes/root_password.py:124
msgid "The password was set by kickstart."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:80
msgid "Status unknown (missing)"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:83
msgid "Status unknown"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:85
msgid "Unmanaged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:88
#: pyanaconda/ui/gui/spokes/network.py:92
msgid "Unavailable"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:90
msgid "Firmware missing"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:96
msgid "Cable unplugged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:98
msgid "Disconnected"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:103
msgid "Connecting"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:105
msgid "Authentication required"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:107
msgid "Connected"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:109
msgid "Disconnecting"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:111
msgid "Connection failed"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:276
msgid "Ethernet"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:277
msgid "Wireless"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:278
#: pyanaconda/ui/gui/spokes/network.glade:15
msgid "Bond"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:279
#: pyanaconda/ui/gui/spokes/network.glade:27
msgid "VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:280
#: pyanaconda/ui/gui/spokes/network.glade:23
msgid "Team"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:281
#: pyanaconda/ui/gui/spokes/network.glade:19
msgid "Bridge"
msgstr ""

#. TRANSLATORS: ethernet cable is unplugged
#: pyanaconda/ui/gui/spokes/network.py:872
msgid "unplugged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:966
#: pyanaconda/ui/gui/spokes/network.glade:519
#: pyanaconda/ui/gui/spokes/network.glade:794
msgid "IPv4 Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:967
#: pyanaconda/ui/gui/spokes/network.glade:494
#: pyanaconda/ui/gui/spokes/network.glade:806
#: pyanaconda/ui/gui/spokes/network.glade:1373
msgid "IPv6 Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:969
#: pyanaconda/ui/gui/spokes/network.py:971
#: pyanaconda/ui/gui/spokes/network.glade:1361
msgid "IP Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1059
#, python-format
msgid "%d Mb/s"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1238
msgid "WEP"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1241
msgid "WPA"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1244
msgid "WPA2"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1248
msgid "Enterprise"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1253 widgets/src/SpokeSelector.c:81
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:152
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:594
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:928
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1248
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1538
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:535
msgid "None"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1433
#: pyanaconda/ui/gui/spokes/network.glade:2442
msgid "Authentication required by wireless network"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1434
#, python-format
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network '%(network_id)s'."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1448
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1456
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Key:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1466
msgid "User name: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1477
msgid "Identity: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1482
msgid "Private key password: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1542
msgctxt "GUI|Spoke"
msgid "_Network & Host Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.py:55
msgctxt "GUI|Spoke"
msgid "_Root Password"
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.py:156
msgid "Disabled, set password to enable."
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.py:161
msgid "Root password is set"
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.py:163
msgid "Root password is not set"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:61
msgctxt "GUI|Spoke"
msgid "_Software Selection"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:231
msgid "Error checking software dependencies"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:310
#: pyanaconda/payload/__init__.py:1658
msgid "Red Hat CDN requires registration"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:326
msgid "Invalid environment specified in kickstart"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:339
msgid "Selected environment is not valid"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:505
msgid "Error checking software dependencies.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:619
msgid ""
"The software marked for installation has the following errors.  This is "
"likely caused by an error with your installation source.  You can quit the "
"installer, change your software source, or change your software selections."
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:624
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:625
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Modify Software Source"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:626
msgctxt "GUI|Software Selection|Error Dialog"
msgid "Modify _Selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/blivet_gui.py:65
msgctxt "GUI|Spoke"
msgid "_Blivet-GUI Partitioning"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:134
msgid "Please wait... software metadata still loading."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:137
#, python-format
msgid ""
"Your current <b>%(product)s</b> software selection requires <b>%(total)s</b>"
" of available space, including <b>%(software)s</b> for software and "
"<b>%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:146
#, python-format
msgid ""
"Your current <a href=\"\" title=\"%(tooltip)s\"><b>%(product)s</b> software "
"selection</a> requires <b>%(total)s</b> of available space, including "
"<b>%(software)s</b> for software and <b>%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:196
#, python-format
msgid "%s The disks you've selected have the following amounts of free space:"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:206
#, python-format
msgid ""
"<b>You don't have enough space available to install %s</b>.  You can shrink "
"or remove existing partitions via our guided reclaim space tool, or you can "
"adjust your partitions on your own in the custom partitioning interface."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:230
#, python-format
msgid ""
"  You don't have enough space available to install <b>%(product)s</b>, even "
"if you used all of the free space available on the selected disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:242
#, python-format
msgid ""
"<b>You don't have enough space available to install %(productName)s</b>, "
"even if you used all of the free space available on the selected disks.  You"
" could add more disks for additional space, modify your software selection "
"to install a smaller version of <b>%(productName)s</b>, or quit the "
"installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:267
msgctxt "GUI|Spoke"
msgid "Installation _Destination"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:389
msgid ""
"Partitioning method changed - planned storage configuration changes will be "
"cancelled."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:473
msgid "Saving storage configuration..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:479
msgid "Passphrase for autopart encryption not specified."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:495
#: pyanaconda/ui/gui/spokes/storage.py:510
msgid "Failed to save storage configuration..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:515
msgid "Unexpected storage error"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:571 pyanaconda/format_dasd.py:169
msgid "Formatting DASDs"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:721
msgid "Error checking storage configuration.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:724
msgid ""
"Warning checking storage configuration.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:771
#, python-format
msgid ""
"FCP device %(hba_id)s\n"
"WWPN %(wwpn)s\n"
"LUN %(lun)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:774
#, python-format
msgid "NVDIMM device %(namespace)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:783
#, python-format
msgid "%s free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:857
#, python-format
msgid "%(count)d disk selected; %(capacity)s capacity; %(free)s free"
msgid_plural "%(count)d disks selected; %(capacity)s capacity; %(free)s free"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/storage.py:916
msgid ""
"You have chosen to skip boot loader installation. Your system may not be "
"bootable."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1120
msgid "There was a problem with your disk selection. Click here for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1217
msgid "You'll be able to make space available during custom partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1237
msgid ""
"The following errors were encountered when checking your disk selection. You"
" can modify your selection or quit the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1242
#: pyanaconda/ui/gui/spokes/storage.py:1263
msgctxt "GUI|Storage|Error Dialog"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1243
msgctxt "GUI|Storage|Error Dialog"
msgid "_Modify Disk Selection"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1258
msgid ""
"The following errors were encountered when checking your storage "
"configuration.  You can modify your storage layout or quit the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1264
msgctxt "GUI|Storage|Error Dialog"
msgid "_Modify Storage Layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1278
msgid ""
"The following warnings were encountered when checking your storage "
"configuration.  These are not fatal, but you may wish to make changes to "
"your storage layout."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1283
msgctxt "GUI|Storage|Warning Dialog"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:417
msgctxt "GUI|Spoke"
msgid "_Time & Date"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:556
msgid "Restoring hardware time..."
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:572
msgid "Invalid timezone"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:746
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:760
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:946
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:749
msgid "PM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:748
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:757
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:945
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:948
msgid "AM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1109
msgid "You need to set up networking first if you want to use NTP"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1113
msgid "You have no working NTP server configured"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:49
#, python-format
msgid "%s to switch layouts."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:275
msgctxt "GUI|Spoke"
msgid "_Keyboard"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:376
msgid "Testing layouts configuration not available."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:384
msgid "Getting list of layouts..."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:438
msgid ""
"Keyboard layouts are not supported when using VNC.\n"
"However the settings will be used after the installation."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:448
msgid "Layout switching not configured."
msgstr ""

#: pyanaconda/ui/gui/spokes/language_support.py:61
msgctxt "GUI|Spoke"
msgid "_Language Support"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:59
#, python-format
msgid ""
"The RAID level you have selected (%(level)s) requires more disks (%(min)d) "
"than you currently have selected (%(count)d)."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:63
#, python-format
msgid "CONFIGURE %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:64
#, python-format
msgid ""
"Please create a name for this %(container_type)s and select at least one "
"disk below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:69
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:70
msgid "Volume Group"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:69
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:70
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Volume Group:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:71
msgid "Volume"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:71
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Volume:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:131
msgid "Cannot relabel already existing file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:136
msgid "Cannot set label on file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:138
msgid "Unacceptable label format for file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:148
msgid "That mount point is already in use. Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:150
msgid "Please enter a valid mount point."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:152
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:164
msgid "That mount point is invalid. Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:305
msgid "container"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:305
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "container"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:401
#, python-format
msgid ""
"%s may be a system boot partition! Deleting it may break other operating "
"systems. Are you sure you want to delete it?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:403
#, python-format
msgid "Are you sure you want to delete all of the data on %s?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:405
#, python-format
msgid ""
"Are you sure you want to delete all of the data on %s, including snapshots "
"and/or subvolumes?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:705
msgid "Invalid container name"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:468
msgid "DATA"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:477
#: pyanaconda/ui/categories/system.py:29
msgid "SYSTEM"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:521
#, python-format
msgid ""
"You haven't created any mount points for your %(product)s %(version)s "
"installation yet.  You can:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:531
msgctxt "GUI|Custom Partitioning|Autopart Page"
msgid "_Click here to create them automatically."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:557
msgid "Create new mount points by clicking the '+' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:565
msgid ""
"Or, assign new mount points to existing partitions after selecting them "
"below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:570
msgctxt "GUI|Custom Partitioning|Autopart Page"
msgid "_New mount points will use the following partitioning scheme:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:133
#, python-format
msgid ""
"<b>%(count)d disk; %(size)s capacity; %(free)s free space</b> (unpartitioned"
" and in file systems)"
msgid_plural ""
"<b>%(count)d disks; %(size)s capacity; %(free)s free space</b> "
"(unpartitioned and in file systems)"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:196
msgctxt "GUI|Selected Disks Dialog"
msgid "_Do not install boot loader"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:199
#: pyanaconda/ui/gui/spokes/lib/cart.glade:202
msgctxt "GUI|Selected Disks Dialog"
msgid "_Set as Boot Device"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/detailederror.py:47
msgctxt "GUI|Detailed Error Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:38
#, python-format
msgid "You have provided a weak passphrase: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:39
msgid "Passphrases do not match."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.py:52
#, python-format
msgid "%(description)s (%(deviceName)s)"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.py:56
#, python-format
msgid "%(deviceName)s on %(container)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.py:89
#: pyanaconda/ui/lib/entropy.py:93
msgid "minute"
msgid_plural "minutes"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:56
msgid "Preserve"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:57
msgid "Shrink"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:58
msgid "Delete"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:85
#, python-format
msgid "Installation requires a total of <b>%s</b> for system data."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:118
msgid "This device contains the installation source."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:164
#, python-format
msgid "%(freeSize)s of %(devSize)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:171
msgid "Not resizeable"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:195
msgid "Free space"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:217
msgid ""
"You can remove existing file systems you no longer need to free up space for"
" this installation.  Removing a file system will permanently delete all of "
"the data it contains."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:223
msgid ""
"There is also free space available in pre-existing file systems.  While it's"
" risky and we recommend you back up your data first, you can recover that "
"free disk space and make it available for this installation below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:233
#, python-format
msgid "<b>%(count)s disk; %(size)s reclaimable space</b> (in file systems)"
msgid_plural ""
"<b>%(count)s disks; %(size)s reclaimable space</b> (in file systems)"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:240
#, python-format
msgid "Total selected space to reclaim: <b>%s</b>"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:455
#: pyanaconda/ui/gui/spokes/lib/resize.py:460
#: pyanaconda/ui/gui/spokes/lib/resize.glade:269
msgctxt "GUI|Reclaim Dialog"
msgid "Delete _all"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:457
msgctxt "GUI|Reclaim Dialog"
msgid "Preserve _all"
msgstr ""

#: pyanaconda/ui/categories/customization.py:29
msgid "CUSTOMIZATION"
msgstr ""

#: pyanaconda/ui/categories/software.py:29
msgid "SOFTWARE"
msgstr ""

#: pyanaconda/ui/categories/__init__.py:46
msgid "DEFAULT TITLE"
msgstr ""

#: pyanaconda/ui/categories/localization.py:29
msgid "LOCALIZATION"
msgstr ""

#: pyanaconda/ui/categories/user_settings.py:29
msgid "USER SETTINGS"
msgstr ""

#: pyanaconda/ui/lib/space.py:35
#, python-format
msgid ""
"Not enough space in file systems for the current software selection. An "
"additional %s is needed."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:56
msgid "The system needs more random data entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:65
msgid "Entropy can be increased by typing randomly on keyboard"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:66
#, python-format
msgid ""
"After %d minutes, the installation will continue regardless of the amount of"
" available entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:90
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%] "
"(%(rem)d %(min)s remaining)"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:100
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%]"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:105
msgid "Enough entropy gathered, please stop typing."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:108
#, python-format
msgid "Giving up, time (%d minutes) ran out."
msgstr ""

#: pyanaconda/ui/lib/disks.py:118
#, python-format
msgid ""
"You selected disk %(selected)s, which contains devices that also use "
"unselected disk %(unselected)s. You must select or de-select these disks as "
"a set."
msgid_plural ""
"You selected disk %(selected)s, which contains devices that also use "
"unselected disks %(unselected)s. You must select or de-select these disks as"
" a set."
msgstr[0] ""

#: pyanaconda/core/util.py:793
msgid "No host url"
msgstr ""

#: pyanaconda/core/util.py:814
msgid "malformed URL, cannot parse it."
msgstr ""

#: pyanaconda/core/util.py:830
msgid "URL has no host component"
msgstr ""

#: pyanaconda/core/constants.py:81
msgid "Start VNC"
msgstr "Entamar VNC"

#: pyanaconda/core/constants.py:82
msgid "Use text mode"
msgstr "Usar mou testu"

#: pyanaconda/core/constants.py:151
msgid ""
"Support for this hardware has been removed in this major release. Please "
"check the removed functionality section of the release notes."
msgstr ""

#: pyanaconda/core/constants.py:156
msgid ""
"This hardware (or a combination thereof) is not supported by Red Hat. For "
"more information on supported hardware, please refer to "
"http://www.redhat.com/hardware."
msgstr ""

#: pyanaconda/core/constants.py:161
msgid ""
"Simultaneous Multithreading (SMT) technology can provide performance "
"improvements for certain workloads, but introduces several publicly "
"disclosed security issues. You have the option of disabling SMT, which may "
"impact performance. If you choose to leave SMT enabled, please read "
"https://red.ht/rhel-smt to understand your potential risks and learn about "
"other ways to mitigate these risks."
msgstr ""

#: pyanaconda/core/constants.py:169
msgid ""
"Simultaneous Multithreading (SMT) may improve performance for certain "
"workloads, but introduces several publicly disclosed security issues. You "
"can disable SMT, which may impact performance. Please read https://red.ht"
"/rhel-smt to understand potential risks and learn about ways to mitigate "
"these risks."
msgstr ""

#: pyanaconda/core/constants.py:185
msgid "The password is empty."
msgstr ""

#: pyanaconda/core/constants.py:186
msgid "The passphrase is empty."
msgstr ""

#: pyanaconda/core/constants.py:189
msgid "The passwords do not match."
msgstr ""

#: pyanaconda/core/constants.py:190
msgid "The passphrases do not match."
msgstr ""

#: pyanaconda/core/constants.py:193
msgid "The passwords you entered were different. Please try again."
msgstr ""

#: pyanaconda/core/constants.py:194
msgid "The passphrases you entered were different. Please try again."
msgstr ""

#: pyanaconda/core/constants.py:200
msgid "The password is too short"
msgstr ""

#: pyanaconda/core/constants.py:201
msgid "The passphrase is too short"
msgstr ""

#: pyanaconda/core/constants.py:204
msgid "The password you have provided is weak."
msgstr ""

#: pyanaconda/core/constants.py:205
msgid "The passphrase you have provided is weak."
msgstr ""

#: pyanaconda/core/constants.py:208
msgid "The password you have provided is weak:"
msgstr ""

#: pyanaconda/core/constants.py:209
msgid "The passphrase you have provided is weak:"
msgstr ""

#: pyanaconda/core/constants.py:211
msgid "Press <b>Done</b> again to use the password anyway."
msgstr ""

#: pyanaconda/core/constants.py:213
msgid ""
"The password you have provided contains non-ASCII characters. You may not be"
" able to switch between keyboard layouts when typing it."
msgstr ""

#: pyanaconda/core/constants.py:214
msgid ""
"The passphrase you have provided contains non-ASCII characters. You may not "
"be able to switch between keyboard layouts when typing it."
msgstr ""

#: pyanaconda/core/constants.py:216
msgid "You will have to press <b>Done</b> twice to confirm it."
msgstr ""

#: pyanaconda/core/constants.py:217
msgid "Press <b>Done</b> to continue."
msgstr ""

#: pyanaconda/core/constants.py:222
msgid "Empty"
msgstr ""

#: pyanaconda/core/constants.py:223
msgid "Too short"
msgstr ""

#: pyanaconda/core/constants.py:224
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:182
msgid "Weak"
msgstr ""

#: pyanaconda/core/constants.py:225
msgid "Fair"
msgstr ""

#: pyanaconda/core/constants.py:226
msgid "Good"
msgstr ""

#: pyanaconda/core/constants.py:227
msgid "Strong"
msgstr ""

#: pyanaconda/core/constants.py:229
msgid "Hide password."
msgstr ""

#: pyanaconda/core/constants.py:230
msgid "Show password."
msgstr ""

#: pyanaconda/core/constants.py:279
msgid "Probing storage..."
msgstr ""

#: pyanaconda/core/constants.py:280
msgid "Testing availability..."
msgstr ""

#: pyanaconda/core/constants.py:281
msgid "Downloading package metadata..."
msgstr ""

#: pyanaconda/core/constants.py:282
msgid "Downloading group metadata..."
msgstr ""

#: pyanaconda/core/constants.py:285
msgid "Anaconda Installer"
msgstr ""

#: pyanaconda/format_dasd.py:165
msgid "Nothing to format"
msgstr ""

#: pyanaconda/format_dasd.py:173
msgid "Probing storage"
msgstr ""

#: pyanaconda/exception.py:109
#, python-format
msgid ""
"The installation was stopped due to what seems to be a problem with your hardware. The exact error message is:\n"
"\n"
"%s.\n"
"\n"
" The installer will now terminate."
msgstr ""

#: pyanaconda/exception.py:113
msgid "Hardware error occurred"
msgstr ""

#: pyanaconda/exception.py:170
#, python-format
msgid ""
"\n"
"The installation was stopped due to an error which occurred while running in non-interactive cmdline mode. Since there cannot be any questions in cmdline mode, edit your kickstart file and retry installation. \n"
"The exact error message is: \n"
"\n"
"%s. \n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/exception.py:178
#, python-format
msgid ""
"\n"
"Running in cmdline mode, no interactive debugging allowed.\n"
"The exact error message is: \n"
"\n"
"%s.\n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/bootloader.py:253
msgid "/boot file system"
msgstr ""

#: pyanaconda/bootloader.py:411
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following raid "
"levels: %(raid_level)s."
msgstr ""

#: pyanaconda/bootloader.py:419
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following metadata "
"versions: %(metadata_versions)s."
msgstr ""

#: pyanaconda/bootloader.py:427
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following device "
"types: %(types)s."
msgstr ""

#: pyanaconda/bootloader.py:443
#, python-format
msgid "%(name)s must have one of the following disklabel types: %(types)s."
msgstr ""

#: pyanaconda/bootloader.py:455
#, python-format
msgid "%(desc)s cannot be of type %(type)s."
msgstr ""

#: pyanaconda/bootloader.py:461
#, python-format
msgid "%(desc)s must be mounted on one of %(mountpoints)s."
msgstr ""

#: pyanaconda/bootloader.py:473
#, python-format
msgid "%(desc)s must be between %(min)d and %(max)d MB in size"
msgstr ""

#: pyanaconda/bootloader.py:479
#, python-format
msgid "%(desc)s must not be smaller than %(min)dMB."
msgstr ""

#: pyanaconda/bootloader.py:488
#, python-format
msgid "%(desc)s must not be larger than %(max)dMB."
msgstr ""

#: pyanaconda/bootloader.py:506
#, python-format
msgid "%(desc)s must be within the first %(max_end)s of the disk."
msgstr ""

#: pyanaconda/bootloader.py:516
#, python-format
msgid "%s must be on a primary partition."
msgstr ""

#: pyanaconda/bootloader.py:601
msgid ""
"Boot loader stage1 device cannot be on an iSCSI disk which is not configured"
" in iBFT."
msgstr ""

#: pyanaconda/bootloader.py:659 pyanaconda/bootloader.py:773
#, python-format
msgid "%s cannot be on an encrypted block device."
msgstr ""

#: pyanaconda/bootloader.py:725
#, python-format
msgid ""
"%(bootloader_stage2_description)s cannot be on an iSCSI disk which is not "
"configured in iBFT."
msgstr ""

#: pyanaconda/bootloader.py:732
#, python-format
msgid "%(desc)s cannot be of type %(type)s"
msgstr ""

#: pyanaconda/bootloader.py:1423
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not. A drive failure in %(stage2dev)s "
"could render the system unbootable."
msgstr ""

#: pyanaconda/bootloader.py:1430
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not part of this array. The stage1 "
"boot loader will only be installed to a single drive."
msgstr ""

#: pyanaconda/bootloader.py:1733
#, python-format
msgid ""
"%(deviceName)s may not have enough space for grub2 to embed core.img when "
"using the %(fsType)s file system on %(deviceType)s"
msgstr ""

#: pyanaconda/network.py:116
msgid "Host name cannot be None or an empty string."
msgstr ""

#: pyanaconda/network.py:119
msgid "Host name must be 255 or fewer characters in length."
msgstr ""

#: pyanaconda/network.py:122
msgid ""
"Host names can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.',"
" parts between periods must contain something and cannot start or end with "
"'-'."
msgstr ""

#: pyanaconda/network.py:1674
msgid "Connecting..."
msgstr ""

#: pyanaconda/network.py:1676
msgid "Disconnecting..."
msgstr ""

#: pyanaconda/network.py:1697
#, python-format
msgid "Wired (%(interface_name)s) connected"
msgstr ""

#: pyanaconda/network.py:1700
#, python-format
msgid "Wireless connected to %(access_point)s"
msgstr ""

#: pyanaconda/network.py:1703
#, python-format
msgid "Bond %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1707
#, python-format
msgid "Team %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1711
#, python-format
msgid "Bridge %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1721
#, python-format
msgid "VLAN %(interface_name)s (%(parent_device)s, ID %(vlanid)s) connected"
msgstr ""

#: pyanaconda/network.py:1738
#, python-format
msgid "Connected: %(list_of_interface_names)s"
msgstr ""

#: pyanaconda/network.py:1740
msgid "Not connected"
msgstr ""

#: pyanaconda/network.py:1743 pyanaconda/ui/gui/spokes/network.glade:2051
msgid "No network devices available"
msgstr ""

#: pyanaconda/vnc.py:79
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr "%(productName)s %(productVersion)s instalación"

#: pyanaconda/vnc.py:148
#, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr "Instalación de %(productName)s %(productVersion)s nel equipu %(name)s"

#: pyanaconda/vnc.py:167
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "Intentando coneutase col veceru vnc nel sirvidor %s..."

#: pyanaconda/vnc.py:181
msgid "Connected!"
msgstr "¡Coneutáu!"

#: pyanaconda/vnc.py:184
msgid "Will try to connect again in 15 seconds..."
msgstr "Va intentar coneutase dempués de 15 segundos..."

#: pyanaconda/vnc.py:191
#, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] "¡Doime por vencíu dempués de %d intentos de coneutame!\n"
msgstr[1] "¡Doime por vencíu dempués de %d intentos de coneutame!\n"

#: pyanaconda/vnc.py:199
msgid "Attempting to start vncconfig"
msgstr ""

#: pyanaconda/vnc.py:212
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""
"Por favor coneuta manualmente'l veceru vnc a %s pa entamar la instalación"

#: pyanaconda/vnc.py:214
#, python-format
msgid ""
"Please manually connect your vnc client to IP-ADDRESS:%s to begin the "
"install. Switch to the shell (Ctrl-B 2) and run 'ip addr' to find the IP-"
"ADDRESS."
msgstr ""

#: pyanaconda/vnc.py:219
msgid "Starting VNC..."
msgstr "Aniciando VNC..."

#: pyanaconda/vnc.py:256
msgid "The VNC server is now running."
msgstr "El sirvidor VNC ta executándose agora"

#: pyanaconda/vnc.py:260
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""
"\n"
"\n"
"Esbillasti coneutate a un visor vnc en mou sirvidor.\n"
"Pa esto nun necesita poner una contraseña. Si pones una,\n"
"va usase en casu de que la conexón col visor de vnc\n"
"nun tenga éxitu.\n"
"\n"

#: pyanaconda/vnc.py:265
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=PASSWORD boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:269
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""
"\n"
"\n"
"Puedes escoyer executar vnc con una contraseña.\n"
"\n"

#: pyanaconda/vnc.py:271
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""
"\n"
"\n"
"Fallu Desconocíu. Albortando\n"
"\n"

#: pyanaconda/vnc.py:293
msgid ""
"VNC password must be six to eight characters long.\n"
"Please enter a new one, or leave blank for no password."
msgstr ""

#: pyanaconda/product.py:66
#, python-format
msgid "%(productName)s %(productVersion)s INSTALLATION"
msgstr ""

#: pyanaconda/product.py:70
#, python-format
msgid "New %(name)s %(version)s Installation"
msgstr ""

#: pyanaconda/platform.py:52
msgid "boot loader device"
msgstr ""

#: pyanaconda/platform.py:54
msgid "RAID Device"
msgstr ""

#: pyanaconda/platform.py:55
msgid "First sector of boot partition"
msgstr ""

#: pyanaconda/platform.py:122 pyanaconda/platform.py:236
#: pyanaconda/platform.py:252
msgid "Master Boot Record"
msgstr ""

#: pyanaconda/platform.py:129
msgid ""
"You must include at least one MBR- or GPT-formatted disk as an install "
"target."
msgstr ""

#: pyanaconda/platform.py:146
msgid "EFI System Partition"
msgstr ""

#: pyanaconda/platform.py:152
msgid ""
"For a UEFI installation, you must include an EFI System Partition on a GPT-"
"formatted disk, mounted at /boot/efi."
msgstr ""

#: pyanaconda/platform.py:166
msgid "Apple EFI Boot Partition"
msgstr ""

#: pyanaconda/platform.py:169
msgid ""
"For a UEFI installation, you must include a Linux HFS+ ESP on a GPT-"
"formatted disk, mounted at /boot/efi."
msgstr ""

#: pyanaconda/platform.py:201
msgid "PReP Boot Partition"
msgstr ""

#: pyanaconda/platform.py:203
msgid ""
"You must include a PReP Boot Partition within the first 4GiB of an MBR- or "
"GPT-formatted disk."
msgstr ""

#: pyanaconda/platform.py:215
msgid "Apple Bootstrap Partition"
msgstr ""

#: pyanaconda/platform.py:218
msgid ""
"You must include an Apple Bootstrap Partition on an Apple Partition Map-"
"formatted disk."
msgstr ""

#: pyanaconda/platform.py:235
msgid "DASD"
msgstr ""

#: pyanaconda/platform.py:237
msgid "zFCP"
msgstr ""

#: pyanaconda/platform.py:242
msgid ""
"You must include at least one MBR- or DASD-formatted disk as an install "
"target."
msgstr ""

#: pyanaconda/platform.py:256
msgid "You must include at least one MBR-formatted disk as an install target."
msgstr ""

#: pyanaconda/pwpolicy.py:144
#, python-format
msgid "policy name required for %s"
msgstr ""

#: pyanaconda/pwpolicy.py:153
#, python-format
msgid "A %(command)s with the name %(policyName)s has already been defined."
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:96
#, python-format
msgid "Receiving objects: %(percent)d%% (%(fetched)d/%(requested)d) %(bytes)s"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:102
msgid "Writing objects"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:189
#, python-format
msgid "Starting pull of %(branchName)s from %(source)s"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:219
#, python-format
msgid "Preparing deployment of %s"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:239
#, python-format
msgid "Deployment starting: %s"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:242
#, python-format
msgid "Deployment complete: %s"
msgstr ""

#: pyanaconda/payload/dnfpayload.py:245
#, python-format
msgid ""
"Downloading %(total_files)s RPMs, %(downloaded)s / %(total_size)s "
"(%(percent)d%%) done."
msgstr ""

#: pyanaconda/payload/dnfpayload.py:1033
msgid "Starting package installation process"
msgstr ""

#: pyanaconda/payload/dnfpayload.py:1054
msgid "Downloading packages"
msgstr ""

#: pyanaconda/payload/dnfpayload.py:1067
msgid "Preparing transaction from installation source"
msgstr "Tresnando la trensaición dende la fonte d'instalación."

#: pyanaconda/payload/dnfpayload.py:1080
#, python-format
msgid "Installing %s"
msgstr ""

#: pyanaconda/payload/dnfpayload.py:1083
#, python-format
msgid "Configuring %s"
msgstr ""

#: pyanaconda/payload/dnfpayload.py:1086
#, python-format
msgid "Verifying %s"
msgstr ""

#: pyanaconda/payload/livepayload.py:104 pyanaconda/payload/livepayload.py:125
msgid "Installing software"
msgstr ""

#: pyanaconda/payload/livepayload.py:271 pyanaconda/payload/livepayload.py:279
#, python-format
msgid "Downloading %(url)s (%(pct)d%%)"
msgstr ""

#: pyanaconda/payload/livepayload.py:424
msgid "Checking image checksum"
msgstr ""

#: pyanaconda/payload/__init__.py:1656
msgid "Failed to set up installation source"
msgstr ""

#: pyanaconda/payload/__init__.py:1657
msgid "Error downloading package metadata"
msgstr ""

#: widgets/src/SpokeSelector.c:82
msgid "New Selector"
msgstr ""

#: widgets/src/StandaloneWindow.c:52
msgctxt "GUI|Standalone Navigation"
msgid "_Quit"
msgstr ""

#: widgets/src/StandaloneWindow.c:53
msgctxt "GUI|Standalone Navigation"
msgid "_Continue"
msgstr ""

#: widgets/src/BaseWindow.c:147
msgid "DISTRIBUTION INSTALLATION"
msgstr ""

#: widgets/src/BaseWindow.c:148
msgid "SPOKE NAME"
msgstr ""

#: widgets/src/BaseWindow.c:149
msgid "PRE-RELEASE / TESTING"
msgstr ""

#: widgets/src/LayoutIndicator.c:33
#, c-format
msgid "Current layout: '%s'. Click to switch to the next layout."
msgstr ""

#: widgets/src/LayoutIndicator.c:34
#, c-format
msgid "Current layout: '%s'. Add more layouts to enable switching."
msgstr ""

#: widgets/src/MountpointSelector.c:84
msgid "0 GB"
msgstr ""

#: widgets/src/SpokeWindow.c:55
msgctxt "GUI|Spoke Navigation"
msgid "_Done"
msgstr ""

#: widgets/src/DiskOverview.c:97
msgid "New Device"
msgstr ""

#: widgets/src/DiskOverview.c:99 widgets/src/DiskOverview.c:100
msgid "0 MB"
msgstr ""

#: pyanaconda/ui/gui/main.glade:24
msgctxt "GUI|Error Dialog"
msgid "_Exit Installer"
msgstr ""

#: pyanaconda/ui/gui/main.glade:72
msgid "Error"
msgstr "Fallu"

#: pyanaconda/ui/gui/main.glade:94
msgctxt "GUI|Main"
msgid "_No"
msgstr ""

#: pyanaconda/ui/gui/main.glade:108
msgctxt "GUI|Main"
msgid "_Yes"
msgstr ""

#: pyanaconda/ui/gui/main.glade:151
msgid ""
"Are you sure you wish to quit the\n"
"installation process?"
msgstr ""

#: pyanaconda/ui/gui/main.glade:175
msgid "Quit"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade:8
#: pyanaconda/ui/gui/hubs/progress.glade:260
msgid "CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade:88
msgid "Preparing to install"
msgstr "Tresnando la instalación"

#: pyanaconda/ui/gui/hubs/progress.glade:146
msgctxt "GUI|Progress"
msgid "_Finish configuration"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade:200
msgctxt "GUI|Progress"
msgid "_Reboot"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade:8
#: pyanaconda/ui/gui/hubs/summary.glade:145
msgid "INSTALLATION SUMMARY"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade:81
msgctxt "GUI|Summary"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade:97
msgctxt "GUI|Summary"
msgid "_Begin Installation"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade:128
msgid "We won't touch your disks until you click 'Begin Installation'."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:27
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Cancel & add more disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:43
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Reclaim space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:77
#: pyanaconda/ui/gui/spokes/storage.glade:295
msgid "INSTALLATION OPTIONS"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:94
#: pyanaconda/ui/gui/spokes/storage.glade:311
msgid ""
"Here we'll describe how much space is needed for the current software "
"selection."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:120
#: pyanaconda/ui/gui/spokes/storage.glade:337
msgid "disk free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:136
#: pyanaconda/ui/gui/spokes/storage.glade:352
msgid "Free space available for use."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:152
#: pyanaconda/ui/gui/spokes/storage.glade:368
msgid "fs free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:168
#: pyanaconda/ui/gui/spokes/storage.glade:384
msgid "Free space unavailable but reclaimable from existing partitions."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:191
#: pyanaconda/ui/gui/spokes/storage.glade:407
msgid "Here we'll describe what your options are."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:222
msgid "Need Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:245
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Quit installer"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:261
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:438
msgid "No Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:450
#: pyanaconda/ui/gui/spokes/storage.glade:1135
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:67
msgid "INSTALLATION DESTINATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:512
msgid "Device Selection"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:531
msgid ""
"Select the device(s) you'd like to install to.  They will be left untouched "
"until you click on the main menu's \"Begin Installation\" button."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:550
msgid "Local Standard Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:614
#: pyanaconda/ui/gui/spokes/storage.glade:754
msgid "Disks left unselected here will not be touched."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:633
msgid "Specialized & Network Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:704
msgctxt "GUI|Storage"
msgid "_Add a disk..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:721
msgid "Add Specialized Disk"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:781
msgid "Storage Configuration"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:801
msgctxt "GUI|Storage"
msgid "A_utomatic"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:818
msgctxt "GUI|Storage"
msgid "_Custom"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:837
msgctxt "GUI|Storage"
msgid "Advanced Custom (_Blivet-GUI)"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:868
msgctxt "GUI|Storage"
msgid "I would like to _make additional space available."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:900
msgid "Encryption"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:930
msgctxt "GUI|Storage"
msgid "_Encrypt my data."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:946
msgid "You'll set a passphrase next."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:996
msgid "Storage Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:1027
msgid "summary"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:1060
msgctxt "GUI|Storage"
msgid "_Full disk summary and boot loader..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:1099
msgctxt "GUI|Storage"
msgid "_Refresh..."
msgstr ""

#: pyanaconda/ui/gui/spokes/language_support.glade:33
msgid "LANGUAGE SUPPORT"
msgstr ""

#: pyanaconda/ui/gui/spokes/language_support.glade:78
msgid "Select additional language support to be installed:"
msgstr ""

#: pyanaconda/ui/gui/spokes/language_support.glade:186
#: pyanaconda/ui/gui/spokes/welcome.glade:333
msgid "Type here to search."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:74
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:543
msgid "RAID0"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:78
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:547
msgid "RAID1"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:82
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:551
msgid "RAID4"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:86
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:555
msgid "RAID5"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:90
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:559
msgid "RAID6"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:94
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:563
msgid "RAID10"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:193
msgid "Add a new mount point."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:194
msgctxt "GUI|Custom Partitioning"
msgid "Add"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:209
msgid "Remove the selected mount point(s)."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:210
msgctxt "GUI|Custom Partitioning"
msgid "Remove"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:225
msgid "Reload storage configuration from disk."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:226
msgctxt "GUI|Custom Partitioning"
msgid "Refresh"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:291
#: pyanaconda/ui/gui/spokes/custom_storage.glade:1098
#: pyanaconda/ui/gui/spokes/custom_storage.glade:1243
#: pyanaconda/ui/gui/spokes/custom_storage.glade:1330
msgid "Selected Device"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:309
#: pyanaconda/ui/gui/spokes/custom_storage.glade:1116
#: pyanaconda/ui/gui/spokes/custom_storage.glade:1261
#: pyanaconda/ui/gui/spokes/custom_storage.glade:1348
msgid "Device description"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:370
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "Mount _Point:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:417
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "Desired _Capacity:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:468
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Label:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:515
msgid "Device(s):"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:544
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Modify..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:576
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:631
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Device _Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:675
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Encrypt"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:717
msgctxt "GUI|Custom Partitioning|Configure"
msgid "File S_ystem:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:761
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Ref_ormat"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:817
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Volume Group:"
msgstr ""

#. check_accelerators: configureButton
#: pyanaconda/ui/gui/spokes/custom_storage.glade:870
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Modify..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:902
msgctxt "GUI|Custom Partitioning|Configure"
msgid "RA_ID Level:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:959
msgctxt "GUI|Custom Partitioning|Configure"
msgid "LUKS Version:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1030
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Update Settings"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1053
msgid ""
"Note:  The settings you make on this screen will not be applied until you "
"click on the main menu's 'Begin Installation' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1150
msgid ""
"This device is encrypted and cannot be read without a valid passphrase. You "
"may unlock it below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1171
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Passphrase:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1200
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Unlock"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1295
msgid ""
"This device cannot be edited directly. You can remove it or select a "
"different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1443
msgid "AVAILABLE SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1496
msgid "TOTAL SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1536
msgctxt "GUI|Custom Partitioning"
msgid "_Reset All"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:136
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "Search _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:153
msgid "Port / Target / LUN #"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:154
msgid "Target WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:193
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Port:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:219
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Target:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:247
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:289
msgctxt "GUI|Installation Destination|Filter|Search|WWID"
msgid "_WWID:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:339
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "Search Res_ults:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:386
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1438
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1761
#: pyanaconda/ui/gui/spokes/installation_source.glade:1038
#: pyanaconda/ui/gui/spokes/lib/resize.glade:163
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:235
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:732
#: pyanaconda/ui/gui/spokes/lib/cart.glade:152
msgid "Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:400
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:597
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:800
msgid "WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:414
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:814
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1148
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1466
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1789
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:248
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:745
#: pyanaconda/ui/gui/spokes/lib/cart.glade:164
msgid "Capacity"
msgstr "Capacidá"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:428
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:596
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:842
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:930
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1176
msgid "Interconnect"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:442
msgid "Model"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:456
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1541
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1817
msgid "LUN"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:470
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1831
msgid "Port"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:484
msgid "Target"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:498
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:595
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:828
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:929
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1162
msgid "Vendor"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:512
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1249
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1424
msgid "Namespace"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:526
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1250
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1452
msgid "Mode"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:553
msgctxt "GUI|Installation Destination|Filter"
msgid "Searc_h"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:578
msgctxt "GUI|Installation Destination|Filter|Multipath"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:635
msgctxt "GUI|Installation Destination|Filter|Multipath|Vendor"
msgid "Show Only _Devices From:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:676
msgctxt "GUI|Installation Destination|Filter|Multipath|Interconnect"
msgid "Show Only _Devices With:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:717
msgctxt "GUI|Installation Destination|Filter|Multipath|WWID"
msgid "Show Only _Devices Containing:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:856
msgid "Paths"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:886
msgctxt "GUI|Installation Destination|Filter"
msgid "_Multipath Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:912
msgctxt "GUI|Installation Destination|Filter|Other"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:931
msgid "ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:969
msgctxt "GUI|Installation Destination|Filter|Other|Vendor"
msgid "Show Only _Devices From:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1010
msgctxt "GUI|Installation Destination|Filter|Other|Interconnect"
msgid "Show Only _Devices With:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1051
msgctxt "GUI|Installation Destination|Filter|Other|ID"
msgid "Show Only _Devices Containing:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1134
msgid "Identifier"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1206
msgctxt "GUI|Installation Destination|Filter"
msgid "_Other SAN Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1232
msgctxt "GUI|Installation Destination|Filter|NVDIMM Devices"
msgid "Filter B_y:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1288
msgctxt "GUI|Installation Destination|Filter|NVDIMM Devices|Mode"
msgid "Show Only Devices in _Mode:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1305
msgid "Sector"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1306
msgid "Memory"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1333
msgctxt "GUI|Installation Destination|Filter|NVDIMM Devices|Namespace"
msgid "_Namespace:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1496
msgctxt "GUI|Installation Destination|Filter"
msgid "_NVDIMM Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1522
msgctxt "GUI|Installation Destination|Filter|zSeries"
msgid "Filter B_y:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1539
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1747
msgid "CCW"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1540
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1803
msgid "WWPN"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1578
msgctxt "GUI|Installation Destination|Filter|zSeries|CCW"
msgid "_CCW:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1621
msgctxt "GUI|Installation Destination|Filter|zSeries|WWPN"
msgid "_WWPN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1664
msgctxt "GUI|Installation Destination|Filter|zSeries|LUN"
msgid "_LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1775
#: pyanaconda/ui/gui/spokes/lib/summary.glade:155
msgid "Type"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1861
msgctxt "GUI|Installation Destination|Filter"
msgid "_zSeries Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1884
msgctxt "GUI|Installation Destination|Filter"
msgid "_Add zFCP LUN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1899
msgctxt "GUI|Installation Destination|Filter"
msgid "Add EC_KD DASD..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1914
msgctxt "GUI|Installation Destination|Filter"
msgid "Add _iSCSI Target..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1929
msgctxt "GUI|Installation Destination|Filter"
msgid "Add FCo_E SAN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1944
msgctxt "GUI|Installation Destination|Filter"
msgid "Reconfigure _NVDIMM..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1959
msgctxt "GUI|Installation Destination|Filter"
msgid "Refresh _List"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.glade:10
msgid "SOFTWARE SELECTION"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.glade:65
msgid "Base Environment"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.glade:83
msgid "Additional software for Selected Environment"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:108
msgctxt "GUI|Date and Time|NTP"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:122
msgctxt "GUI|Date and Time|NTP"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:147
msgid "Add and mark for usage NTP servers"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:169
msgid "New NTP Server"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:188
msgid "Add NTP Server"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:208
msgid "This URL refers to a pool of NTP servers"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:254
msgid "Pool"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:267
msgid "Working"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:275
msgid "Use"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:303
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:517
msgid "Configure NTP"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:324
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:932
msgid "TIME & DATE"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:374
msgctxt "GUI|Date and Time"
msgid "_Region:"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:404
msgid "Region"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:421
msgctxt "GUI|Date and Time"
msgid "_City:"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:451
msgid "City"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:479
msgctxt "GUI|Date and Time"
msgid "_Network Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:497
msgid "Use Network Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:584
msgid "Hours"
msgstr ""

#. TRANSLATORS: This is the separator between hours and minutes, like in HH:MM
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:597
msgctxt "GUI|Date and Time"
msgid ":"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:617
msgid "Minutes"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:635
msgid "Hour Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:653
msgid "Hour Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:671
msgid "Minutes Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:689
msgid "Minutes Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:733
msgid "AM/PM Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:769
msgid "AM/PM Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:820
msgctxt "GUI|Date and Time"
msgid "24-_hour"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:838
msgctxt "GUI|Date and Time"
msgid "_AM/PM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:897
msgid "Set Date & Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.glade:8
#: pyanaconda/ui/gui/spokes/root_password.glade:194
msgid "ROOT PASSWORD"
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.glade:53
msgctxt "GUI|Password"
msgid "_Root Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.glade:67
msgctxt "GUI|Password"
msgid "_Confirm:"
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.glade:107
#: pyanaconda/ui/gui/spokes/user.glade:191
msgid "Confirm Password"
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.glade:141
#: pyanaconda/ui/gui/spokes/user.glade:254
msgid "empty password"
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.glade:162
msgid ""
"The root account is used for administering the system.  Enter a password for"
" the root user."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:29
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:43
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Open"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:98
msgctxt "GUI|Software Source|Media Check Dialog"
msgid "_Done"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:131
msgid "MEDIA VERIFICATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:158
msgid "Verifying media, please wait..."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:230
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:244
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:273
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Enable HTTP Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:305
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Proxy Host:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:322
msgid "<span size=\"small\"><b>Example:</b> squid.mysite.org:3128</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:355
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Use Authentication"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:381
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "User _name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:398
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "Pass_word:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:480
msgid "INSTALLATION SOURCE"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:537
msgid "Which installation source would you like to use?"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:551
msgctxt "GUI|Software Source"
msgid "_Auto-detected installation media:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:577
msgid "Device:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:589
msgid "Label:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:599
#: pyanaconda/ui/gui/spokes/installation_source.glade:735
msgctxt "GUI|Software Source"
msgid "_Verify"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:623
msgctxt "GUI|Software Source"
msgid "Installation media via SE/_HMC"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:641
msgctxt "GUI|Software Source"
msgid "Red Hat _CDN"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:660
msgctxt "GUI|Software Source"
msgid "_ISO file:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:688
msgctxt "GUI|Software Source"
msgid "D_evice:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:719
msgctxt "GUI|Software Source"
msgid "_Choose an ISO"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:759
msgctxt "GUI|Software Source"
msgid "_On the network:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:823
msgctxt "GUI|Software Source"
msgid "_Proxy setup..."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:840
#: pyanaconda/ui/gui/spokes/installation_source.glade:1312
msgctxt "GUI|Software Source"
msgid "URL type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:855
#: pyanaconda/ui/gui/spokes/installation_source.glade:1297
msgid "repository URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:856
#: pyanaconda/ui/gui/spokes/installation_source.glade:1298
msgid "mirrorlist"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:857
#: pyanaconda/ui/gui/spokes/installation_source.glade:1299
msgid "metalink"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:884
msgctxt "GUI|Software Source"
msgid "N_FS mount options:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:899
#: pyanaconda/ui/gui/spokes/installation_source.glade:900
msgid "This field is optional."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:933
msgid "Updates"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:960
msgctxt "GUI|Software Source"
msgid ""
"Don't install the latest available software _updates. Install the default "
"versions provided by the installation source above."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:984
msgid "Additional repositories"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1025
msgid "Enabled"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1070
#: pyanaconda/ui/gui/spokes/installation_source.glade:1071
msgid "Add a new repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1072
msgctxt "GUI|Software Source"
msgid "A_dd"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1089
#: pyanaconda/ui/gui/spokes/installation_source.glade:1090
msgid "Remove the selected repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1091
msgctxt "GUI|Software Source"
msgid "_Remove"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1108
#: pyanaconda/ui/gui/spokes/installation_source.glade:1109
msgid "Revert to the previous list of repositories."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1110
msgctxt "GUI|Software Source"
msgid "Rese_t"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1140
#: pyanaconda/ui/gui/spokes/installation_source.glade:1141
msgid "URL for the repository, without protocol."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1153
#: pyanaconda/ui/gui/spokes/installation_source.glade:1154
msgid "URL of proxy in the form of protocol://host:[port]"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1166
#: pyanaconda/ui/gui/spokes/installation_source.glade:1167
msgid "Optional proxy user name."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1179
msgid "Optional proxy password."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1193
msgctxt "GUI|Software Source"
msgid "_Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1210
#: pyanaconda/ui/gui/spokes/installation_source.glade:1211
msgid "Protocol for the repository URL."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1230
msgctxt "GUI|Software Source"
msgid "Pro_xy URL:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1247
msgctxt "GUI|Software Source"
msgid "U_ser name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1264
msgctxt "GUI|Software Source"
msgid "Pass_word:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1281
#: pyanaconda/ui/gui/spokes/installation_source.glade:1282
msgid "Name of the repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:40
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:54
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:85
msgid "ADD iSCSI STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:114
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:142
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Target IP Address:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:170
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "iSCSI _Initiator Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:187
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:760
msgid "No credentials (discovery authentication disabled)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:188
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:761
msgid "CHAP pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:189
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:762
msgid "CHAP pair and a reverse pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:203
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Discovery Authentication Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:218
msgid ""
"<span size=\"small\"><b>Example:</b> iqn.2012-09.com.example:diskarrays-"
"sn-a8675309</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:263
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:277
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:389
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:403
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:417
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP User _name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:431
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:465
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Start"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:502
msgid "Discovering iSCSI targets.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:541
msgid "Discovery login rejected."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:556
msgid ""
"The following error occurred discovering iSCSI targets.  Please double check"
" your authorization information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:579
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Retry"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:619
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Bind targets to network interfaces This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:695
msgid "Node Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:706
msgid "Interface"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:717
msgid "Portal"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:745
msgctxt "GUI|Advanced Storage|iSCSI|Login"
msgid "_Node Login Authentication Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:763
msgid "Use the credentials from discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:797
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:811
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:923
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:937
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:951
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "_Reverse CHAP User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:965
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1014
msgctxt "GUI|Advanced Storage|iSCSI|Login|Login"
msgid "_Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1069
msgid "Node login failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1084
msgid ""
"The following error occurred logging into the selected iSCSI node.  Please "
"double check your authorization information and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1107
msgctxt "GUI|Advanced Storage|iSCSI|Login|Retry"
msgid "Retry _Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:82
msgid "ADD zFCP STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:105
msgid ""
"To use zFCP disks, you must provide the device number, WWPN, and LUN "
"configured for the device."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:132
msgctxt "GUI|Advanced Storage|zFCP|Device Number"
msgid "_Device number:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:159
msgid "WWPN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:177
msgctxt "GUI|Advanced Storage|zFCP|Start Discovery"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:214
msgid "Discovering zFCP devices.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:253
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:215
msgid "Device discovery failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:268
msgid ""
"The following error occurred discovering zFCP devices.  Please double check "
"your configuration information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:291
msgctxt "GUI|Advanced Storage|zFCP|Retry Discovery"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:346
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:307
msgid "Device discovery succeeded."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:373
msgid "LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:30
msgctxt "GUI|Advanced Storage|NVDIMM|Reconfigure"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:77
msgid ""
"The following NVDIMM devices selected from the list will be reconfigured to "
"sector mode:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:92
msgctxt "GUI|Advanced Storage|NVDIMM|Reconfigure"
msgid "_Sector size:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:110
msgctxt "GUI|Advanced Storage|NVDIMM|Reconfigure"
msgid "Start _Reconfiguration"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:146
msgid "Reconfiguring NVDIMM devices. This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:185
msgid "Device reconfiguration failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:212
msgid "The following error occurred reconfiguring NVDIMM devices:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:259
msgid "Devices reconfiguration succeeded."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:285
msgid ""
"Rescanning disks after reconfiguration.\n"
"This may take a moment ..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:347
msgid "Warning: Existing data on reconfigured devices will be lost."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:23
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:56
msgid ""
"Please select the network interface which is connected to your FCoE switch."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:70
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_NIC:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:92
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use _DCB"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:108
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use auto _VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:144
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Add FCoE Disk(s)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:187
msgid "Error adding FCoE SAN."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:215
msgid ""
"The following error occurred while attempting to add an FCoE SAN.  Please "
"check your configuration and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:25
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:39
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:70
msgid "ADD DASD STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:93
msgid "To use DASD disks, you must provide the device number."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:120
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Device number:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:139
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:176
msgid "Discovering DASD devices.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:230
msgid ""
"The following error occurred discovering DASD devices.  Please double check "
"your configuration information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:253
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:322
msgid "The added DASD can be found under \"Local Standard Disks.\""
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:36
msgctxt "GUI|Advanced User"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:50
msgctxt "GUI|Advanced User"
msgid "_Save Changes"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:82
msgid "ADVANCED USER CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:105
msgctxt "GUI|Advanced User"
msgid "Home _directory:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:140
msgid "User and Group IDs"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:160
msgctxt "GUI|Advanced User"
msgid "Specify a _user ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:177
msgctxt "GUI|Advanced User"
msgid "Specify a _group ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:237
msgid "Group Membership"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:265
msgctxt "GUI|Advanced User"
msgid "_Add user to the following groups:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:298
msgid "Example:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:315
msgid "wheel, my-team (1245), project-x (29935)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:348
msgid "Tip:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:367
msgid ""
"You may input a comma-separated list of group names and group IDs here. "
"Groups that do not already exist will be created; specify their GID in "
"parentheses. "
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:10
#: pyanaconda/ui/gui/spokes/user.glade:350
msgid "CREATE USER"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:62
msgctxt "GUI|User"
msgid "_Full name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:80
msgctxt "GUI|User"
msgid "_User name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:100
msgid "Full Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:117
#: pyanaconda/ui/gui/spokes/network.glade:1582
msgid "User Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:132
msgctxt "GUI|User"
msgid "_Password"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:150
msgctxt "GUI|User"
msgid "_Confirm password"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:205
msgid ""
"<b>Tip:</b> Keep your user name shorter than 32 characters and do not use "
"spaces."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:215
msgctxt "GUI|User"
msgid "_Require a password to use this account"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:273
msgctxt "GUI|User"
msgid "_Make this user administrator"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:293
msgctxt "GUI|User"
msgid "_Advanced..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:35
msgid "Add device"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:49
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:63
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Add"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:92
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Select the type of device you wish to add"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:165
#: pyanaconda/ui/gui/spokes/network.glade:2277
msgid "NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:204
msgid ""
"Please use the live desktop environment's tools for customizing your network"
" configuration.  You can set the host name here."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:417
msgid "Slaves"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:441
#: pyanaconda/ui/gui/spokes/network.glade:830
#: pyanaconda/ui/gui/spokes/network.glade:1397
msgid "DNS"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:469
#: pyanaconda/ui/gui/spokes/network.glade:818
#: pyanaconda/ui/gui/spokes/network.glade:1385
msgid "Default Route"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:544
#: pyanaconda/ui/gui/spokes/network.glade:782
#: pyanaconda/ui/gui/spokes/network.glade:1229
msgid "Speed"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:569
#: pyanaconda/ui/gui/spokes/network.glade:770
msgid "Hardware Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:594
msgid "Parent"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:606
msgid "VLAN ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:654
msgctxt "GUI|Network|Wired"
msgid "_Configure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:943
msgid "Security"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:955
msgctxt "GUI|Network|Wireless"
msgid "_Network Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:969
msgid "Network Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:981
msgid "Security Key"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1065
msgctxt "GUI|Network|Wireless"
msgid "_Use as Hotspot..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1082
msgctxt "GUI|Network|Wireless"
msgid "_Stop Hotspot..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1099
msgctxt "GUI|Network|Wireless"
msgid "_Configure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1217
msgid "IMEI"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1241
msgid "Provider"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1424
msgctxt "GUI|Network|Mobile Broadband"
msgid "_Configure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1534
msgid "VPN Type"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1546
msgid "Gateway"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1558
msgid "Group Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1570
msgid "Group Password"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1690
msgctxt "GUI|Network|VPN"
msgid "_Configure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1812
msgctxt "GUI|Network|Proxy"
msgid "_Method"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1837
msgctxt "GUI|Network|Proxy"
msgid "_Configuration URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1862
msgctxt "GUI|Network|Proxy"
msgid "HTTP _Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1876
msgctxt "GUI|Network|Proxy"
msgid "H_TTPS Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1890
msgctxt "GUI|Network|Proxy"
msgid "_FTP Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1904
msgctxt "GUI|Network|Proxy"
msgid "_Socks Host"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2094
msgid "Network Config Box"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2110
msgctxt "GUI|Network"
msgid "Unloc_k"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2143
msgctxt "GUI|Network"
msgid "Airplane Mode"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2163
msgid "More Network Config Box"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2183
msgctxt "GUI|Network"
msgid "_Host Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2212
msgctxt "GUI|Network|Hostname"
msgid "_Apply"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2216
msgid "Set as current system host name."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2231
msgid "Current host name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2283
msgid "NETWORK CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2328
msgid ""
"We'll need network access to fetch information about your location and to make software\n"
"updates available for you."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2358
#: pyanaconda/ui/gui/spokes/subscription.glade:83
msgid "Authentication"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2372
msgctxt "GUI|Network|Authentication Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2387
msgctxt "GUI|Network|Authentication Dialog"
msgid "C_onnect"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2459
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network"
msgstr ""

#: pyanaconda/ui/gui/spokes/blivet_gui.glade:8
msgid "BLIVET GUI PARTITIONING"
msgstr ""

#: pyanaconda/ui/gui/spokes/blivet_gui.glade:131
msgctxt "GUI|Blivet Gui Partitioning"
msgid "_Undo last action"
msgstr ""

#: pyanaconda/ui/gui/spokes/blivet_gui.glade:150
msgctxt "GUI|Blivet Gui Partitioning"
msgid "_Reset All"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:23
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "I want to _exit."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:37
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "I want to _proceed."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:68
msgid "This is unstable, pre-release software."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:97
msgid ""
"Notice: This is pre-released software that is intended for development and testing purposes only.  Do *not* use this software for any critical work or for production environments.\n"
"\n"
"By clicking \"I want to proceed\", you understand and accept the risks associated with pre-released software, that you intend to use this for testing and development purposes only and are willing to report any bugs or issues in order to enhance this work.\n"
"\n"
"If you do not understand or accept the risks, then please exit this program by selecting \"I want to exit\" which will reboot your system."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:130
msgid "Beta Warn"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:228
msgid "What language would you like to use during the installation process?"
msgstr "¿Qué llingua quies usar na instalación?"

#: pyanaconda/ui/gui/spokes/welcome.glade:313
msgid "Languages"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:385
msgid "Locales"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:419
msgid "WELCOME"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:15
#: pyanaconda/ui/gui/spokes/keyboard.glade:388
msgid "KEYBOARD LAYOUT"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:56
msgid ""
"Which keyboard layouts would you like to use on this system?  You may move "
"any layout to the top of the list to select it as the default."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:96
msgid ""
"Changes here will only apply to the installed system. Use the desktop's tool"
" to configure the keyboard for the installation process."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:167
msgid "Selected Layouts"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:190
msgctxt "GUI|Keyboard Layout"
msgid "_Add layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:205
msgctxt "GUI|Keyboard Layout"
msgid "_Remove layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:220
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _up"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:235
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _down"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:251
msgctxt "GUI|Keyboard Layout"
msgid "_Preview layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:291
msgctxt "GUI|Keyboard Layout"
msgid "_Test the layout configuration below:"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:331
msgid "Alt + Shift to switch layouts."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:346
msgctxt "GUI|Keyboard Layout"
msgid "_Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:421
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:435
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Add"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:467
msgid "ADD A KEYBOARD LAYOUT"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:483
msgid "You may add a keyboard layout by selecting it below:"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:532
msgid "Available Layouts"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:573
msgid "Add Layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:605
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:619
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:651
msgid "LAYOUT SWITCHING OPTIONS"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:668
msgid ""
"Which combination(s) would you prefer for switching between keyboard "
"layouts?"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:751
msgid "Layout Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:8
msgid "CONNECT TO RED HAT"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:100
msgctxt "GUI|Subscription|Authentication|Account"
msgid "_Account"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:117
msgctxt "GUI|Subscription|Authetication|Activation Key"
msgid "Activation _Key"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:227
msgid "Organization"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:239
msgid "Activation Key"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:263
msgid "key1,key2,..."
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:285
msgid "Purpose"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:311
msgid "Usage"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:323
msgid "Role"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:335
msgid "SLA"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:382
msgctxt "GUI|Subscription|Set System Purpose"
msgid "Set System Purpose"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:399
#: pyanaconda/ui/gui/spokes/subscription.glade:844
msgid "Insights"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:412
msgctxt "GUI|Subscription|Red Hat Insights"
msgid "Connect to Red Hat _Insights"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:453
msgid "Custom base URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:499
msgid "Custom server URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:513
msgid "Use HTTP proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:540
msgid "Location"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:576
msgid "hostname:port"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:642
msgid "Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:665
msgid "The system is currently not registered."
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:678
msgctxt "GUI|Subscription|Register"
msgid "_Register"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:724
msgid "The system has been properly subscribed"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:738
msgctxt "GUI|Subscription|Unregister"
msgid "_Unregister"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:774
msgid "Method"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:791
msgid "System Purpose"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:906
msgid "No subscriptions have been attached to the system"
msgstr ""

#: pyanaconda/ui/gui/spokes/subscription.glade:981
msgid "SYSTEM PURPOSE"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:23
msgctxt "GUI|Passphrase Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:38
msgctxt "GUI|Passphrase Dialog"
msgid "_Save Passphrase"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:65
msgid "DISK ENCRYPTION PASSPHRASE"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:81
msgid ""
"You have chosen to encrypt some of your data. You will need to create a "
"passphrase that you will use to access your data when you start your "
"computer."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:102
msgctxt "GUI|Passphrase Dialog"
msgid "_Passphrase:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:133
msgctxt "GUI|Passphrase Dialog"
msgid "Con_firm:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:275
msgid ""
"Warning: You won't be able to switch between keyboard layouts (from the "
"default one) when you decrypt your disks after install."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:8
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:71
msgid "RESCAN DISKS"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:25
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:39
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:87
msgid ""
"You can remove or insert additional disks at this time and press 'Rescan "
"Disks' below for the changes to take effect."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:118
msgid ""
"<b>Warning:</b> All storage changes made using the installer will be lost "
"when you press 'Rescan Disks'."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:143
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Rescan Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:176
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:212
msgid "Scanning disks.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:223
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while this scan completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:254
msgid "Disk rescan complete."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:269
msgid ""
"Pressing 'OK' below will take you to the disk selection screen where you "
"will need to re-select your disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:8
msgid "UNFORMATTED DASDS"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:23
msgctxt "GUI|DAD Format Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:37
msgctxt "GUI|DAD Format Dialog"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:70
msgid ""
"The following unformatted or LDL DASDs have been detected on your system. "
"You can choose to format them now with dasdfmt or cancel to leave them "
"unformatted. Unformatted DASDs cannot be used during installation."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:86
msgid "Unformatted DASDs Detected"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:136
msgid ""
"<b>Warning</b>: All storage changes made using the installer will be lost "
"when you choose to format."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:160
msgctxt "GUI|DAD Format Dialog"
msgid "_Format with dasdfmt"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:209
msgid ""
"This may take a moment. You may <a href=\"\">go back to the main menu</a> to"
" complete other installation options while formatting completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:248
msgid "Disk formatting complete."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:55
msgctxt "GUI|Reclaim Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:70
msgctxt "GUI|Reclaim Dialog"
msgid "_Reclaim space"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:103
msgid "RECLAIM DISK SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:120
msgid "Description goes here."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:174
msgid "File System"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:185
msgid "Reclaimable Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:196
#: pyanaconda/ui/gui/spokes/lib/summary.glade:144
msgid "Action"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:221
msgctxt "GUI|Reclaim Dialog"
msgid "_Preserve"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:237
msgctxt "GUI|Reclaim Dialog"
msgid "_Delete"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:253
msgctxt "GUI|Reclaim Dialog"
msgid "_Shrink"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:367
msgid "Reclaim"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:21
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:37
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Delete It"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:129
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:144
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Select"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:175
msgid "CONFIGURE MOUNT POINT"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:191
msgid "Select one or more disks this device may reside on."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:222
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:719
#: pyanaconda/ui/gui/spokes/lib/cart.glade:139
msgid "Description"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:261
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:758
#: pyanaconda/ui/gui/spokes/lib/cart.glade:177
msgid "Free"
msgstr "Llibre"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:321
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:336
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Add mount point"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:370
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "Desired _Capacity:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:387
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "Mount _Point:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:405
msgid "ADD A NEW MOUNT POINT"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:423
msgid ""
"More customization options are available\n"
"after creating the mount point below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:458
msgid ""
"That mount point is already in\n"
"use.  Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:482
msgid "eg: \"20 GB\", \"500mb\" (minus the quotation marks)"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:539
msgid "Single"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:587
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:601
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Save"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:633
msgid "CONFIGURE CONTAINER"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:649
msgid ""
"Please create a name for this container and select at least one disk below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:667
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:786
msgid "RAID Level:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:815
msgid "Encrypt"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:856
msgid "Please enter a valid name."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:886
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "Si_ze policy:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:901
msgid "Automatic"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:902
msgid "As large as possible"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:903
msgid "Fixed"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/unsupported_hardware.glade:23
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/unsupported_hardware.glade:37
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Continue"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/unsupported_hardware.glade:122
msgid "Unsupported Hardware"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/detailederror.glade:25
msgid "An unknown error occurred during installation.  Details are below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:25
#: pyanaconda/ui/gui/spokes/lib/summary.glade:89
msgid "SUMMARY OF CHANGES"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:42
msgctxt "GUI|Summary Dialog"
msgid "_Cancel & Return to Custom Partitioning"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:57
msgctxt "GUI|Summary Dialog"
msgid "_Accept Changes"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:105
msgid ""
"Your customizations will result in the following changes taking effect after"
" you return to the main menu and begin installation:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:133
msgid "Order"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:166
msgid "Device"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:26
#: pyanaconda/ui/gui/spokes/lib/cart.glade:79
msgid "SELECTED DISKS AND BOOT LOADER"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:45
msgctxt "GUI|Selected Disks Dialog"
msgid "_Close"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:96
msgid "Selected disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:125
msgid "Boot"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:218
msgctxt "GUI|Selected Disks Dialog"
msgid "_Remove"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:251
msgid "Secure Boot"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:267
msgid ""
"Firmware will verify the integrity of the Linux kernel during boot if the "
"Secure Boot is enabled and configured on the machine. Secure Boot is not "
"supported on IBM z14 and earlier models, therefore choose to disable it if "
"you intend to boot the installed system on such models."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:285
msgid "Enabled if supported by the installing machine"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:286
msgid "Enabled unconditionally"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:287
msgid "Disabled"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:308
msgid "Disk summary goes here"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:35
msgid ""
"The system needs better quality of random data, you can improve it by typing"
" randomly on keyboard and moving your mouse. The installation will continue "
"automatically regardless of random data quality when time runs out."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:53
msgid "Random data quality:"
msgstr ""

#: data/liveinst/liveinst.desktop:3
msgid "Install to Hard Drive"
msgstr "Instalar nel Discu Duru"

#: data/liveinst/liveinst.desktop:4
msgid "Install"
msgstr "Instalar"

#: data/liveinst/liveinst.desktop:5
msgid "Install the live CD to your hard disk"
msgstr "Instalar el CD vivu nel discu duru"

#. TRANSLATORS: Icon name, probably should not be translated
#: data/liveinst/liveinst.desktop:11
msgid "anaconda"
msgstr ""

#: data/liveinst/gnome/rhel-welcome.desktop:3
#: data/liveinst/gnome/rhel-welcome.js:62
msgid "Welcome to Red Hat Enterprise Linux"
msgstr ""

#: data/liveinst/liveinst-setup.desktop:4
msgid "Liveinst Setup"
msgstr ""

#: data/liveinst/gnome/rhel-welcome.js:91
msgid "Try RHEL"
msgstr ""

#: data/liveinst/gnome/rhel-welcome.js:107
msgid ""
"You are currently running Red Hat Enterprise Linux from live media.\n"
"You can install Red Hat Enterprise Linux now, or choose \"Install to Hard Drive\" in the Activities Overview at any later time."
msgstr ""

#: data/liveinst/gnome/rhel-welcome.js:125
msgid ""
"You can choose \"Install to Hard Drive\"\n"
"in the Activities Overview at any later time."
msgstr ""

#: data/liveinst/gnome/rhel-welcome.js:128
msgid "Close"
msgstr ""

#: ../data/liveinst/console.apps/liveinst.h:1
msgid "Starting Install to Hard Drive"
msgstr "Aniciando Instalación nel Discu Duru"
